\selectlanguage{spanish}
\chapter{Resumen}

La Biología del Desarrollo estudia la formación de un organismo complejo con cientos de tipos celulares a partir de una única celula, el cigoto. Esto implica procesos de proliferación y diferenciación celular. Los factores de transcripción son clave en la especificación regional y diferenciación celular, delimitando qué genes se activarán en una célula concreta, confiriéndole por tanto su identidad. 

Los genes Hox son una clase de factores de transcripción implicados en desarrollo particularmente importante. Están ampliamente conservados en el reino animal y forman el núcleo del sistema molecular de formación de patrón antero-posterior en todos los animales bilaterales. 

Ya que los genes Hox reconocen secuencias de ADN cortas e indistintas, sus funciones de regulación transcripcional requieren de la participación de cofactores. Los cofactores Hox pertenecen a la clase TALE, un conjunto de factores de transcripción con un homeodominio divergente. En mamíferos, hay tres familias de cofactores TALE Hox : Meis, Prep y Pbx. 

En esta tesis hemos abordado un análisis a escala genómica de los sitios de unión al ADN de la casi totalidad de los cofactores TALE Hox presentes en el embrión de ratón a estadio E11.5. Describimos que Meis y Prep, que se comportan casi indistinguiblemente \textit{in vitro}, reconocen secuencias de ADN muy distintas en el embrión. Mientras que las secuencias unidas por Meis sugieren que actúa principalmente como cofactor Hox, Prep parece unir ADN predominantemente en forma de hetrodímeros con Pbx. 

No sólo sus secuencias de unión tienen aspecto muy distinto. Las demás características de sus sitios de unión son también muy distintas, sugiriendo funciones divergentes. Prep une muchos promotores y está asociado a la activación transcripcional de los genes afectados. En cambio, Meis une ADN en un patrón casi aleatorio con respecto a los sitios de inicio de transcripción. 






\selectlanguage{english}
\chapter{Summary}

Developmental Biology studies the formation of a complex organism with hundreds of cell types from a single cell, the zygote. This involves both proliferation and differentiation. Transcription factors are key players in and cell differentiation, selecting which genes will be active in a particular cell, thus defining its identity. 

The Hox genes are a particularly important class of developmental transcription factors. They are widely conserved among animals and form the core of the molecular system for antero-posterior patterning. Since Hox genes recognize short, very frequent DNA sequences that are very similar for different Hoxes, they require cofactors to effect their transcriptional regulatory functions. The Hox cofactors belong to the TALE family. In mammals, there are three classes of TALE Hox cofactors: Meis, Prep and Pbx. 

In this thesis, we have undertaken a near comprehensive analysis of the genomic binding sites of the TALE Hox cofactors in the E11.5 mouse embryo. We describe that Meis and Prep, which behave very similarly \textit{in vitro} with regards to binding sequences, have very distinct binding preferences in the embryo. While the sequences bound by Meis suggest that is is acting mostly as a Hox cofactor, Prep seems to bind DNA predominantly in heterodimers with Pbx. 

They not only have distinct sequence preferences, but also bind genomic sites with very different characteristics and, likely, functions. Prep, specially when in combination with Pbx, binds a high number of promoters and is associated with transcriptional activation of the bound genes. In contrast, Meis binds DNA with a nearly random distribution relative to transcription start sites. Despite being remote from promoters, the phylogenetic conservation of Meis binding sites is higher than that for any other transcription factor we have found in the literature, except HoxC9, and many of them bear enhancer epigenetic marks in published datasets, suggesting they are very highly conserved enhancer elements. 

Meis, Prep and Pbx binding sites show a suggestive pattern in the four murine Hox clusters, in all of which Meis binds very densely to the 3' portion with a clear restriction to the region 3' of the Hox9 paralog. Hox genes transcript levels are affected in Meis1 and Prep1 loss-of-function models in a collinear and opposite fashion. 

Taken together, our results paint a picture in which Meis and Prep are functionally specialized. While Meis is mainly a cofactor for Hox genes, binding in concert with them and Pbx to many highly conserved non coding elements, Prep binds with Pbx but no participation of Hoxes to promoters and activates the corresponding genes. They often have opposing effects on transcription, for example in the Hox clusters. 




