\chapter{Introduction}


\label{chp:intro}

\section{Intro to TALEs and Hoxes}

\subsection{Hoxes}

\subsubsection{History, Function}

William Bateson defined the term homeosis in 1894 to describe morphological variations in which a part of the body is substituted for another. Homeotic mutations thus affect genes important for establishing body part identity. Several such mutations were discovered during the era of classical genetics in Drosophila. Deletion of Ultrabithorax, for example, results in a shift of the identity of the third thoracic segment into second thoracic, as shown by the presence of wings instead of halteres (\cite{ref}) [more examples?]

Selector gene hypothesis (\cite{Garcia-Bellido1975,Garcia-Bellido1977}) [include? fill]

After the work of many developmental geneticists culminating in Edward B. Lewis' landmark paper (\cite{Lewis1978}) the basic mechanism of segmental specification in Drosophila was extensively understood. There were a number of mutations known to result in anterior to posterior transformations. It was known that these mutations involved a single cluster of linked genes, the \ac{BX-C}. It was known that loss of function mutations produce anteriorization of segmental identity while gains of function result in posteriorization. [kaufman1990:opposite for \ac{ANT-C} (?) \ac{ANT-C} when?]

It was deduced, from the phenotypes and their combinations, that there are two corresponding gradients in the embryo: a proximo-distal gradient in the chromosome and a \ac{BX-C} gene action antero-posterior gradient such that any gene active in a segment is active in all segments posterior to it- this before in situ hybridization was developed [check] or even the actual products of the genes affected where known.

Polycomb was known to be a repressor of all genes in the cluster necessary for proper patterning. It was known that segmental identity is cell-autonomously established. 

The work of Christiane NÃ¼sslein-Volhard and Eric F. Wieschaus expanded in this solid foundation (\cite{Nuesslein-Volhard1980}). They performed an extensive screen that identified many of the genes that would occupy developmental biologists for more than a decade. [eliminate? not really relevant for Hoxes. If not, complete]

After the cloning of the Drosophila \ac{BX-C} (\cite{Bender1983}) and \ac{ANT-C} (\cite{Garber1983,Scott1983}), it was noted that probes for Antennapedia cross-hibridized with several other genes contained in them, indicating sequence similarity (\cite{McGinnis1984}).  The stretch of homology spans 180 base pairs, was called the homeobox and codes for a protein domain called the homeodomain. The homeodomain is even more conserved in sequence than the homeobox and from the very beginning was suspected to be responsible for DNA binding due to its high content in basic aminoacids (see \cite{Gehring1985}).

Understanding of the homeotic selector genes and of animal development in general underwent a revolution when, shortly afterwards, it was discovered that the homeobox is present in vertebrates. In two papers published back to back, Carrasco and co-workers reported cloning of a Xenopus gene with homology to Antennapedia, Ultrabithorax and fushi tarazu (\cite{Carrasco1984}) while McGinnis and co-workers reported the sequence of these three Drosophila genes and, strikingly, the presence of homologs in several animal species far removed philogenetically from insects (\cite{McGinnis1984b}). In a context where no vertebrate genes controlling development were known, this must have been an earth-shattering revelation. Suddenly a system well studied in Drosophila potentially played a role in shaping the animals closest to us. It hinted at a deep underlying layer of similarity uniting all complex animal body plans that had been unsuspected before, and that would be confirmed and explored in the years to come.

The various described genetic functions were grouped into three complementation groups: Ubx, Abd-A and Abd-B (\cite{Sanchez-Herrero1985}[is this the actual description of abd-A?si:kaufman1990. Irrelevant?]).

[Colinearity term earliest use I found:\cite{Lewis1985}]

By the early 1990s, It was well established that the Hox clusters of vertebrates and the homeotic complex of Drosophila were homologous. The colinearity of the domain of gene expression with respect to position along the chromosome was long since described in  Drosophila, and it had been recently reported that in vertebrates there is additional colinearity with timing of gene expression and RA sensitivity. Hox proteins were known to be able to bind DNA and suspected to be transcription factors (reviewed in \cite{Levine1988, McGinnis1992}). 

\begin{figure}[]
  
  \centering
  \label{fig:hoxclusters}
  \includegraphics[width=\textwidth]{figures/HoxClusters}
  \caption[The Organization of Hox Genes]{\textbf{The Organization of Hox Genes.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}

A new stage in the study of Hox genes had started that focused on their molecular function, through the use of homologous recombination in mouse to generate loss of function and in vitro techniques to characterize protein activity. The recent understanding of the high conservation of Hox gene function and organization through most of the animal kingdom would aid by making findings in different model organisms directly translatable. 

[find Ubx ChIP 1990]
[first mouse knockout \cite{Thomas1987}]



[who and when proved molecular function of hoxes as TFs? Levine? in Levine1988 it is suggested, in McGinnis1992 it is established]

[who and when developed ISH?]


\ac{ANT-C}: reviewed in \cite{Kaufman1990}




\subsubsection{Cluster structure, gene structure}


Drosophila homeotic genes are organized in two clusters, the  Bithorax (reviewed in \cite{Lewis1978}) and Antennapedia (reviewed in \cite{Kaufman1990}) complexes. The two of them form the HOM-C. The Drosophila "cluster" was the first to be described, but in many aspects it is quite atypical. Both components are located in the third chromosome of the fly but separated by about 10Mb, more than one third of the total length of the chromosome. They are much longer than their vertebrate counterparts. Several non-Hox-related genes are contained in the \ac{ANT-C} and not all the Hox genes share the same transcriptional orientation. 

In mammals there are 39 Hox genes grouped in four clusters, tightly organized, each containing homologs of different Drosophila Hoxes. There are 13 different sets of homologous genes (paralog groups) but no cluster contains examples of all paralogs. The only paralog groups present in all clusters are 4, 9 and 13, homologs of Drosophila Deformed and Abdominal B. [for figure footnote: The 5' part of the clusters is clearly different evolutionarily: all are homologs to AbdB, etc[complete]] 

For a more extensive comparative analysis of Hox clusters see \cite{Duboule2007}.

All Hox genes share a 180 bp sequence, the homeobox. It codes for a 60aa DNA binding domain, the homeodomain. The homeodomain is an example of helix-turn-helix DNA-binding domain. is formed by three alpha-helices, of which [structure, DNA contact description: find a review]. 

\subsubsection{regulation of Hoxes}

Even before the discovery of the mouse Hox genes, it was already known that Hoxes have complex patterns of auto- and cross-regulation. The most prevalent feature of this cross-regulation is called phenotypic suppression (\cite{Gonzalez-Reyes1990}) or posterior prevalence (\cite{Lufkin1991}) and consists on the ability of more posterior (i.e. 5') Hoxes to suppress the effect of more 3' Hoxes. It explains the tendency of loss of function mutations to result in anteriorization while gain of function mutations usually result in posteriorization.

[auto- and cross- regulation]

Polycomb was described as early as 1978 as a repressor needed for the correct expression of Hox genes (\cite{Lewis1978}). In addition, the trithorax complex is an epigenetic activator of gene expression that also plays a role in the proper expression of Hox genes.[complete]

The first known non-Hox molecules to be described as regulators of Hox gene expression were \ac{RA} (\cite{Simeone1990}), which shifts their expression domains anteriorly and Krox20 (\cite{Swiatek1993}[check this citation]), which plays an important part in establishing their pattern in the rhombomeres of the vertebrate neural tube.

\ac{RA} signalling is mediated by retinoid acid receptors and retinoic X receptors. These are cytoplasmic proteins that translocate to the nucleus in the presence of \ac{RA} and activate gene expression by binding to \acp{RARE}. A number of \acp{RARE} have been described in the Hox clusters \cite{refs}. [expand]

Apart from Krox20, other regulators of Hox gene expression in the rhombomeres are Kreisler (\cite{ref}), Cdx1/2 (\cite[refs]), AP-2 (\cite{refs}), GATA-1 (\cite{ref}), [others?]



miRNAs


\subsubsection{Downstream: the need for cofactors}
[DNA binding]

Hox genes are transcription factors. A simple view of their molecular effect is that they bind DNA and activate or repress the transcription of their nearby genes. However, all of the Hox homeodomains recognize slightly different  variants of the same, very short, DNA sequence. 

They need to coordinate the expression of many targets (few of which are known, though), activated or repressed differentially by different paralogs, despite their very short recognition sequence. This contradiction baffled biologists for [how long? who? not sure about the word...], but the discovery of DNA binding partners attenuated the [controversy no encaja, intensity of the question?]. The Hox cofactors [fill]

[Hox protein regulation of translation: look for the paper].

\subsection{History and general perspective of Hox cofactors}

Pbx was identified in Drosophila in 1990 as \ac{exd} (\cite{Peifer1990}) and proposed as a Hox protein cofactor based on phenotype. In parallel, it was identified as the DNA binding part of the chimeric protein produced by the t(1;19) translocation found in human pre-B cell acute lymphoblastic leukemia (\cite{Kamps1990}). 
Meis1 was identified in 1995 as a proto-oncogene locus in which the ecotropic murine leukemia virus [check] is inserted in BXH-2 mice (\cite{Moskow1995}). \ac{hth} was identified as a Drosophila homolog of Meis1 required for \ac{exd} nuclear localization (\cite{Rauskolb1995, Rieckhof1997}).
Prep1 was identified in 1998 as a component of the urokinase enhancer factor 3 capable of forming a DNA binding-independent heterodimer with Pbx1 \cite{Berthelsen1998}.

\subsection{Philogeny: hoxes, tales}

The homeodomain is present in many genes other than Hoxes. Homologs of homodomain genes have been found as far from vertebrates as plants [Knotted, Yeast mating proteins, others, buscar: mirar \cite{Burglin1997}]. Meis, Prep and Pbx are part of a subgroup of homeodomain proteins characterized by the presence in their homeodomain of a \ac{TALE} that gives name to the class and is situated between alpha-helices 1 and 2 \cite{Bertolino1995, Burglin1997, Mukherjee2007, Moens2006}

\subsection{Structure of the TALE genes and proteins}
	
		There are six distinct subgroups of \ac{TALE} homodomain proteins in bilaterian genomes. Of these, only 3 are known to be Hox cofactors [verify]. These three groups, the PREP, MEIS, and PBC classes, share homologous sequences at their N-terminal ends \cite{Burglin1998}. These sequences, called HM-1 and HM-2 in the MEIS and PREP classes and PBC-A and PBC-B in the PBC class, are exclusive to the three Hox cofactor classes despite their being apart in the \ac{TALE} superclass philogeny (\cite{Mukherjee2007}).
		The PBC class is the most divergent within the \ac{TALE} superclass. Members of this class have a fourth alpha helix 3' to the homeodomain(\cite{Mukherjee2007}). 

\subsection{interactions}

		Pbx was the first of the three cofactors to be shown to bind DNA cooperatively with Hoxes \cite{ref}. Over the years, a number of protein complexes involving two or more partners binding on the surface of DNA or independent of it have been described \textit{in vitro} and [in vivo]. The picture emerging from two decades of in vitro studies is complex. 
		All three classes of \ac{TALE} Hox cofactors have the nominal extension in their homeodomains, but only Meis and Pbx proteins have been shown to bind Hoxes directly [verify Prep hasn't]. Pbx is described in the literature as being the main Hox cofactor \cite{ref}. It binds Hox proteins through its \ac{TALE} and the Hox YPWM motif, present in paralog groups 1 to 10 \cite{refs, Passner1999}. Meis can bind directly Hoxes belonging to groups 9 to 13 through its carboxy-terminal domain [what part of the hoxes?] \cite{ref}. 
		Both Meis and Prep are capable of forming heterodimers with Pbx proteins \cite{ref}. In the case of Meis/\ac{hth}, this binding is required for the nuclear localization of the Pbx/\ac{exd} protein {ref}. 
		In addition to these heterodimeric complexes, a number of trimeric complexes have been reported. In these, generally one of the protein partners does not bind DNA. [expand]

		[other interactions - PKA etc]

\subsection{binding specifities}
		
		The Hox proteins have a short recognition sequence of 4 nucleotides that in most cases resembles TAAT. 
		
		When binding DNA in heterodimers with Pbx the consensus target sequence is TGATNNAT, where the 5' TGAT part is bound by Pbx and the 3' NNAT is bound by the Hox protein. The 2 NN nucleotides differ according to the Hox paralog involved in the binding: the standard view is that TAAT is recognized by anterior Hoxes while posterior Hoxes recognize TTAT.
		
		However, the above described consensus sequences represent the binding site for which a particular Pbx-Hox heterodimer has the most affinity. It may be that the important factor \textit{in vivo} is not affinity, but the selectivity of a particular site that allows it to choose between many Pbx-Hox heterodimers. An example is the Hoxb1 autoregulatory enhancer. It contains several Hox-Pbx binding sites, of which one, R3, is highly specific for HoxB1-Pbx1. In Drosophila, a similar sequence is present in a labial autoregulatory enhancer. Changing the sequence of this lab-\ac{exd} target to TGATTAAT switches the specifity to dfd-\ac{exd} (\cite{Chan1997}). However, the TGATGGAT original sequence is not the maximum affinity consensus for lab-\ac{exd} to begin with (\cite{Mann1998}). The possibility that selectivity is more important than affinity is borne by the fact that many homeodomains, despite having identical highest-affinity targets, differ in their "secondary" target motifs (\cite{Berger2008}).
		
		Several relatively recent studies have applied high throughput techniques to the description of homeodomain binding preferences. (\cite{Slattery2011,Berger2008,Noyes2008})
		
		[TGAT, NNAT, TGACAG, TGATTGACAG, TGATNNAT, others]



\subsection{Expression}
\subsection{Functions, phenotypes}

		Meis1  \cite{Azcoitia2005, Carramolino2010}. [fill] [Other Meises - ask Laura for summary]. Joint overexpression of Meis1 and Hoa7 or Hoxa9 is sufficient to induce myeloid leukemia {ref}.
		Prep1 null, \ac{prepi} \cite{Ferretti2006, Fernandez-Diaz2010, Longobardi2010}. [fill] [Other Preps?]
		Pbx1 deletion results in both hematopoietic \cite{DiMartino2001} and patterning \cite{Selleri et al. 2001} defects in mice.

\section{Regulation of gene expression}
[meter o no meter?]
\subsection{Promoters, RNApolII, TFS}

\subsection{Enhancers and enhancer-binding proteins}

\subsection{Epigenetics: histone marks}

\subsection{Chromatin conformation}

\section{Intro to ChIP-seq}

\subsection{Original description}
The \ac{chip} assay consists of cross-linking of chromatin to preserve the non-covalent interactions between proteins and DNA for biochemical assay. It was first described by \cite{Solomon1988}. For many years its application remained confined to focused application \cite{Mardis2007}.  

The first genome-wide modification of the technique involved microarray assaying of the enriched DNA fragments (ChIP-chip, \cite{Ren2000}). As massive sequencing has gone down in cost, its application to assaying ChIP-enriched DNA fragments has increased. ChIP-seq \cite{Robertson2007} [complete]

[Comparison between chipchip and chipseq: el paper del chipexo]

\subsection{methodology, shortcomings}




[meter Deep homology (eg tinman-nkx2.5, hth-Meis, Pax6-eyeless)?]
