\chapter{Introduction}


\label{chp:intro}

\section{The Hox genes}

\subsection{History of Hox Genes Study}

William Bateson defined the term homeosis in 1894 to describe morphological variations in which a part of the body is substituted for another. Homeotic mutations thus affect genes important for establishing body part identity. Several such mutations were discovered during the era of classical genetics in Drosophila. Deletion of \ac{ubx}, for example, results in a shift of the identity of the third thoracic segment into second thoracic, as shown by the presence of wings instead of halteres (\cite{ref}) [more examples?]

Selector gene hypothesis (\cite{Garcia-Bellido1975,Garcia-Bellido1977}) [include? fill]

After the work of many developmental geneticists culminating in Edward B. Lewis' landmark paper (\cite{Lewis1978}) the basic mechanism of segmental specification in Drosophila was extensively understood. There were a number of mutations known to result in anterior to posterior transformations. It was known that these mutations involved a single cluster of linked genes, the \ac{BX-C}. It was known that loss of function mutations produce anteriorization of segmental identity while gains of function result in posteriorization. [kaufman1990:opposite for \ac{ANT-C} (?) \ac{ANT-C} when?]

It was deduced, from the phenotypes and their combinations, that there are two corresponding gradients in the embryo: a proximo-distal gradient in the chromosome and a \ac{BX-C} gene action antero-posterior gradient such that any gene active in a segment is active in all segments posterior to it- this before in situ hybridization was developed [check] or even the actual products of the genes affected where known.

Polycomb was known to be a repressor of all genes in the cluster necessary for proper patterning. It was known that segmental identity is cell-autonomously established. 

The work of Christiane NÃ¼sslein-Volhard and Eric F. Wieschaus expanded in this solid foundation (\cite{Nuesslein-Volhard1980}). They performed an extensive screen that identified many of the genes that would occupy developmental biologists for more than a decade. [eliminate? not really relevant for Hoxes. If not, complete]

After the cloning of the Drosophila \ac{BX-C} (\cite{Bender1983}) and \ac{ANT-C} (\cite{Garber1983,Scott1983}), it was noted that probes for \ac{antp} cross-hibridized with several other genes contained in them, indicating sequence similarity (\cite{McGinnis1984}).  The stretch of homology spans 180 base pairs, was called the homeobox and codes for a protein domain called the \ac{HD}. The \ac{HD} is even more conserved in sequence than the homeobox and from the very beginning was suspected to be responsible for DNA binding due to its high content in basic aminoacids (see \cite{Gehring1985}).

Understanding of the homeotic selector genes and of animal development in general underwent a revolution when, shortly afterwards, it was discovered that the homeobox is present in vertebrates. In two papers published back to back, Carrasco and co-workers reported cloning of a Xenopus gene with homology to \ac{antp}, \ac{ubx} and \ac{ftz} (\cite{Carrasco1984}) while McGinnis and co-workers reported the sequence of these three Drosophila genes and, strikingly, the presence of homologs in several animal species far removed philogenetically from insects (\cite{McGinnis1984b}). In a context where no vertebrate genes controlling development were known, this must have been an earth-shattering revelation. Suddenly a system well studied in Drosophila potentially played a role in shaping the animals closest to us. It hinted at a deep underlying layer of similarity uniting all complex animal body plans that had been unsuspected before, and that would be confirmed and explored in the years to come.

The various described genetic functions were grouped into three complementation groups: Ubx, Abd-A and Abd-B (\cite{Sanchez-Herrero1985}[is this the actual description of abd-A?si:kaufman1990. Irrelevant?]).

[Colinearity term earliest use I found:\cite{Lewis1985}]

By the early 1990s, It was well established that the Hox clusters of vertebrates and the homeotic complex of Drosophila were homologous. The colinearity of the domain of gene expression with respect to position along the chromosome was long since described in  Drosophila, and it had been recently reported that in vertebrates there is additional colinearity with timing of gene expression and RA sensitivity. Hox proteins were known to be able to bind DNA and suspected to be transcription factors (reviewed in \cite{Levine1988, McGinnis1992}). 

\begin{figure}[]
  
  \centering
  \label{fig:hoxclusters}
  \includegraphics[width=\textwidth]{figures/HoxClusters}
  \caption[The Organization of Hox Genes]{\textbf{The Organization of Hox Genes.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}

A new stage in the study of Hox genes had started that focused on their molecular function, through the use of homologous recombination in mouse to generate loss of function and \textit{in vitro} techniques to characterize protein activity. The recent understanding of the high conservation of Hox gene function and organization through most of the animal kingdom would aid by making findings in different model organisms directly translatable. 

[Ubx ChIP 1990: \cite{Gould1990}]
[first mouse knockout \cite{Thomas1987}]

[who and when proved molecular function of hoxes as TFs? Levine? in Levine1988 it is suggested, in McGinnis1992 it is established]

[who and when developed ISH?]

\ac{ANT-C}: reviewed in \cite{Kaufman1990}

\subsection{Cluster and Gene Structure}


Drosophila homeotic genes are organized in two clusters, the  Bithorax (reviewed in \cite{Lewis1978}) and \ac{antp} (reviewed in \cite{Kaufman1990}) complexes. The two of them form the HOM-C. The Drosophila "cluster" was the first to be described, but in many aspects it is quite atypical. Both components are located in the third chromosome of the fly but separated by about 10Mb, more than one third of the total length of the chromosome. They are much longer than their vertebrate counterparts. Several non-Hox-related genes are contained in the \ac{ANT-C} and not all the Hox genes share the same transcriptional orientation. 

In mammals there are 39 Hox genes grouped in four clusters, tightly organized, each containing homologs of different Drosophila Hoxes. There are 13 different sets of homologous genes (paralog groups) but no cluster contains examples of all paralogs. The only paralog groups present in all clusters are 4, 9 and 13, homologs of Drosophila Deformed and Abdominal B. [for figure footnote: The 5' part of the clusters is clearly different evolutionarily: all are homologs to AbdB, etc[complete], particular pattern of interactions, conservation of 4 paralogs in 9 and 13, ] 

For a more extensive comparative analysis of Hox clusters see \cite{Duboule2007}.

All Hox genes share a 180 bp sequence, the homeobox. The homeobox codes for a 60aa DNA binding domain, the \ac{HD}, related to the helix-turn-helix DNA-binding domain of prokaryotes. It is constituted by a flexible \ac{N-ter} arm and three alpha-helices, of which the third slots into the major groove of DNA and interacts with specific nucleotide residues (\cite{Treisman1992}). The specificity of this interaction is critically affected by \ac{aa} 9 of the recognition helix. (\cite{Treisman1992}). \acp{aa} 6 and 10 also contact DNA and are very conserved across \acp{HD}. Arginines at positions 3 and 5 of the \ac{N-ter} non-helical portion of the \ac{HD} also contact DNA on the core recognition site, but in the the minor groove side. Arginine 5 is almost invariant across \acp{HD} (\cite{Treisman1992}). Arginine 3 is present in Hox groups 2-8 and contributes to a preference for TAAT sites. The presence of a Lysine in position 3 is characteristic of the \ac{Abd-B} subfamily and probably contributes to its preference for TTAT.

The hexapeptide/linker/\ac{N-ter} of the \ac{HD} is an important determinant of DNA specificity and ability to interact with Pbx, as shown by chimeric \ac{HD} experiments (\cite{Chang1996,Phelan1997}). It is present in all Hox non-Abd-like proteins (pralogs 1-8). Paralogs 9 and 10 have an alternative tryptophan-containing motif \ac{N-ter} of the \ac{HD}.

Transcriptional activator/repressor functions are mediated by domains other than the \ac{HD} and differ between paralog groups \cite{ref}.

\subsection{Regulation of the Hox Genes}

Even before the discovery of the mouse Hox genes, it was already known that Hoxes have complex patterns of auto- and cross-regulation. The most prevalent feature of this cross-regulation is called phenotypic suppression (\cite{Gonzalez-Reyes1990}) or posterior prevalence (\cite{Lufkin1991}) and consists on the ability of more posterior (i.e. 5') Hoxes to suppress the effect of more 3' Hoxes. It explains the tendency of loss of function mutations to result in anteriorization while gain of function mutations usually result in posteriorization.

[auto- and cross- regulation]

Polycomb was described as early as 1978 as a repressor needed for the correct expression of Hox genes (\cite{Lewis1978}). In addition, the trithorax complex is an epigenetic activator of gene expression that also plays a role in the proper expression of Hox genes.[complete]

The first known non-Hox molecules to be described as regulators of Hox gene expression were \ac{RA} (\cite{Simeone1990}), which shifts their expression domains anteriorly, and Krox20 (\cite{Swiatek1993}[check this citation]), which plays an important part in establishing their pattern in the rhombomeres of the vertebrate neural tube.

\ac{RA} signalling is mediated by retinoid acid receptors and retinoic X receptors. These are cytoplasmic proteins that translocate to the nucleus in the presence of \ac{RA} and activate gene expression by binding to \acp{RARE}. A number of \acp{RARE} have been described in the Hox clusters \cite{refs}. [expand]

Apart from Krox20, other regulators of Hox gene expression in the rhombomeres are Kreisler (\cite{ref}), Cdx1/2 (\cite[refs]), AP-2 (\cite{refs}), GATA-1 (\cite{ref}), [others?]



miRNAs


\subsection{Downstream: the Need for Cofactors}
[DNA binding]

Hox genes are transcription factors. A simple view of their molecular effect is that they bind DNA and activate or repress the transcription of their nearby genes. However, all of the Hox \acp{HD} recognize slightly different  variants of the same, very short, DNA sequence. 

They need to coordinate the expression of many targets (few of which are known, though), activated or repressed differentially by different paralogs, despite their very short recognition sequence. This contradiction baffled biologists for [how long? who? not sure about the word...], but the discovery of DNA binding partners attenuated the [controversy no encaja, intensity of the question?] (see \cite{Mann1996}). The Hox cofactors [fill]


[Hox protein regulation of translation: look for the paper].

\section{History and general perspective of Hox cofactors}

Pbx was identified in Drosophila in 1990 as \ac{exd} (\cite{Peifer1990}) and proposed as a Hox protein cofactor based on phenotype. In parallel, it was identified as the DNA binding part of the chimeric protein produced by the t(1;19) translocation found in human pre-B cell acute lymphoblastic leukemia (\cite{Kamps1990}). They were recognised as homologs in 1993 (\cite{Rauskolb1993}).
Meis1 was identified in 1995 as a proto-oncogene locus in which the ecotropic murine leukemia virus [check] is inserted in BXH-2 mice (\cite{Moskow1995}). \ac{hth} was identified as a Drosophila homolog of Meis1 required for \ac{exd} nuclear localization (\cite{Rauskolb1995, Rieckhof1997}).
Prep1 was identified in 1998 as a component of the urokinase enhancer factor 3 capable of forming a DNA binding-independent heterodimer with Pbx1 \cite{Berthelsen1998,Berthelsen1998a}.

\subsection{Philogeny: hoxes, tales}

The \ac{HD} is present in many genes other than Hoxes. Homologs of homodomain genes have been found as far from vertebrates as plants [Knotted, Yeast mating proteins, others, buscar: mirar \cite{Burglin1997}]. Meis, Prep and Pbx are part of a subgroup of \ac{HD} proteins characterized by the presence in their \ac{HD} of a \ac{TALE} that gives name to the class and is situated between alpha-helices 1 and 2 \cite{Bertolino1995, Burglin1997, Mukherjee2007, Moens2006}

\subsection{Structure of the TALE genes and proteins}

There are six distinct subgroups of \ac{TALE} homodomain proteins in bilaterian genomes. Of these, only 3 are known to be Hox cofactors [verify]. These three groups, the PREP, MEIS, and PBC classes, share homologous sequences at their \ac{N-ter} ends \cite{Burglin1998}. These sequences, called HM-1 and HM-2 in the MEIS and PREP classes and PBC-A and PBC-B in the PBC class, are exclusive to the three Hox cofactor classes despite their being apart in the \ac{TALE} superclass philogeny (\cite{Mukherjee2007}).

\begin{figure}[]
  
  \centering
  \label{fig:TALE_Hox_protStructure}
  \includegraphics[width=\textwidth]{figures/TALE_Hox_protStructure}
  \caption[Domains of the TALE and Hox proteins]{\textbf{Domains of the TALE and Hox proteins.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}

The PBC class is the most divergent within the \ac{TALE} superclass. Members of this class have a fourth alpha helix 3' to the \ac{HD}(\cite{Mukherjee2007}). 

Residue 50 of the \ac{HD}, the 9\textsuperscript{th} of the 3\textsuperscript{rd} helix, is important for base selection by contacting the major groove of DNA and is a glicine in the PBC class, in contrast to the long side chain \acp{aa} in Hox proteins. [other cofactors?]

[splicing isoforms]

\section{Interactions Between the Hox genes and their TALE cofactors}
\label{sec:interactions}

Pbx was the first of the three cofactors to be shown to bind DNA cooperatively with Hoxes \cite{Chan1994}. Over the years, a number of protein complexes involving two or more partners binding on the surface of DNA or independent of it have been described \textit{in vitro} and \textit{in vivo}. The picture emerging from two decades of in vitro studies is complex. 

\subsection{Two-way Interactions}

All three classes of \ac{TALE} Hox cofactors have the nominal extension in their \acp{HD}, but only Meis and Pbx proteins have been shown to bind Hoxes directly [verify Prep hasn't]. Pbx is described in the literature as being the main Hox cofactor \cite{ref}. Its binding to Hox proteins requires its \ac{HD} plus the following 15 \acp{aa} and the Hox \ac{HD} plus hexapeptide (also known as YPWM motif), present in paralog groups 1 to 8 \cite{Chang1995, Passner1999}. Paralog groups 9 and 10 can also interact with Pbx through an alternative ANW motif located just 5' to their \ac{HD} (\cite{Chang1996, Shen1997a}). Paralog groups 11 to 13 can not bind Pbx1a despite most of them containing tryptophan residues 5' to their \ac{HD}. 

Crystallographic structure determination of a Exd-Ubx-DNA complex (\cite{Passner1999}) and a Pbx1-HoxB1-DNA complex (\cite{Piper1999}) showed that Pbx and the Hox component of the heterodimer bind DNA on opposite sides of the helix. The \ac{N-ter} linker (20 \acp{aa} long in HoxB1 and 8 \acp{aa} long in Ubx) of the Hox protein extends over the DNA so that the hexapeptide contacts Pbx directly, in an hydrophobic pocket partially formed by the DNA recognition helix Pbx \ac{HD} and by its \ac{TALE}. There is also potential for the \ac{N-ter} of the recognition helix of Pbx to interact with the \ac{C-ter} of the Hox recognition helix, since the two helices are oriented head-to-tail and their ends are close. Later determination of an \ac{Abd-A} homolog, HoxA9, in complex with Pbx1 and DNA showed very similar arrangement of the proteins in the DNA (\cite{LaRonde-LeBlanc2003}). Despite HoxA9's hexapeptide having a radically diferent backbone structure from Ubx's or HoxB1's, the critical tryptophan residue slots into Pbx1's hydrophobic pocket in the exact same position. One of the few differences between the determined structures is that HoxA9's \ac{HD} N-terminal linker is ordered, unlike Ubx's or HoxB1's.

[two more structures: in \cite{Joshi2007} Scr-Exd bound to specific Scr target or consensus target]

\begin{figure}[]
  
  \centering
  \label{fig:PbxHoxCrystal}
  \includegraphics[width=\textwidth]{figures/PbxHoxCrystal}
  \caption[Hox-Pbx-DNA Complex structure]{\textbf{Hox-Pbx-DNA Complex structure.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}

Meis1 can bind directly Hoxes belonging to groups 9 to 13 through its \ac{C-ter} portion. This interaction greatly stabilizes Meis DNA binding, which may occur as homodimers in the absence of Hoxes or Pbx. The homeodomain of Hoxes is necessary for binding to Meis, but further \acp{aa} \ac{N-ter} to the Hox \ac{HD} are necessary to stabilize the binding, at least in the case of HoxA9. The part of Meis1 required for Hox binding is located somewhere within the \ac{HD} or the 38 following \acp{aa}, which are common to Meis1a and Meis1b. The two main Meis1 isoforms do not seem to differ in their protein or DNA binding characteristics (\cite{Shen1997}). [Is hth restricted to Ubx binding???]

Both Meis and Prep are capable of forming heterodimers with Pbx proteins \cite{ref}. In the case of Meis/\ac{hth}, this binding is required for the nuclear localization of the Pbx/\ac{exd} protein \cite{ref}.

Pbx and Meis can interact independently of DNA, and \textit{in vivo} immunopurified Pbx-containing complexes avidly bind the joint Pbx-Meis binding target (see \ref{sec:specificites}), suggesting that most of the Pbx in the cell is complexed with Meis (\cite{Chang1997}) or Prep. This interaction is required for Pbx/exd to translocate to the nucleus (\cite{Rieckhof1997}) and for stability of the Meis/hth protein (\cite{Abu-Shaar1998}). The Pbx Meis-interacting surface is contained within its first 88 \acp{aa}, \ac{N-ter} to the point of fusion in E2a-Pbx1 chimeras and within the PBC-A domain. The Meis Pbx-interacting surface is located between \acp{aa} 30 and 60 (\cite{Chang1997}). The fact that Meis and Pbx interact through their flexible \ac{N-ter} arms results in non-strict position and orientation requirements for their DNA targets (\cite{Jacobs1999}).

Prep binds Pbx independently of DNA through its \ac{N-ter} conserved domains and the Pbx part deleted in E2a-Pbx (\cite{Berthelsen1998}), just like Meis. 

see \cite{Mann1996}

\begin{figure}[]
  
  \centering
  \label{fig:TALE_Hox_interactions}
  \includegraphics[width=0.5\textwidth]{figures/TALE_Hox_interactions}
  \caption[Summary of TALE-Hox interactions]{\textbf{Summary of TALE-Hox interactions.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}


\subsection{Trimeric Complexes}

In addition to the heterodimeric complexes, a number of trimeric complexes have been reported. In these, generally one of the protein partners does not bind DNA. 

Meis and Pbx can form triple protein complexes with at least some of the Hox proteins. Meis1 can form ternary complexes with any of the Pbx's and HoxA9 \textit{in vitro} independently of DNA. Addition of Meis greatly enhances the  Pbx-HoxA9 interaction, which is weak in the absence of DNA (\cite{Shen1999}).  In this study it was also shown that the three proteins colocalize \textit{in vivo} in nuclear speckles, but surprisingly fail to activate or repress a transcriptional reporter bearing their consensus DNA target.

Meis has also been shown to form trimeric complexes with Pbx and HoxB1 (\cite{Jacobs1999}). These heterotrimers bound the HoxB1 ARE and the HoxB2 r4 enhancer sequences and required intact Pbx-Hox and Meis half-sites to bind DNA, but probably not to assemble. Formation of complexes was checked to be dependent of the heterodimerization motifs of each protein component, implying that Pbx is the hinge subunit that interacts with both Meis and Hoxb1 at the same time. Addition of the three proteins resulted in cooperative transcriptional activation in a transient transfection reporter assay and \textit{in vivo} lacZ staining that depended on the motifs for Pbx-Hox and Meis binding, in contrast to \cite{Shen1999}. 



% GGGGG sequence present between the OCTA and HEXA motifs in both r4enh and b1ARE

Prep can form triple complexes with Pbx and a Hox protein. Co-transfection of Prep1, Pbx1 and Hoxb1 resulted in additional activation of a b1-ARE reporter over Pbx1 and Hoxb1 alone (\cite{Berthelsen1998}). This additional activation was independent of Prep1 DNA binding ability but required its ability to bind Pbx.

[other interactions - PKA etc]

\section{DNA Binding Specifities}
\label{sec:specificites}
[\cite{Levine1988} proposes in vitro transcription as a new method applicable to homeobox-containing genes. "it had been previously shown that fly and frog homeobox proteins bind to specific DNA sequence motifs"]
		
The first homeobox proteins to be shown to be sequence-specific \acp{TF} were \ac{ftz}, \ac{en} and \ac{eve} (\cite{Desplan1988,Hoey1988,Hoey1988a}). Two types of binding consensus sequence were described: TCAATTAAAT and TCAGCACCG.
		
\ac{ubx} \ac{chip} showed binding to sites containing a TAAT core (\cite{Gould1990}). This study also showed that sites containing very similar sequences can result in upregulation or downregulation of transcripts, so the sign of transcriptional regulation is probably not encoded in the core target sequence.

%%%Binding site selection paper:
%Science. 1990 Nov 23;250(4984):1104-10.
%Differences and similarities in DNA-binding preferences of MyoD and E2A protein complexes revealed by binding site selection.
%Blackwell TK, Weintraub H.		

\ac{exd} was shown to bind cooperatively with \ac{ubx} to sequences in a \textit{dpp} enhancer (\cite{Chan1994}). In this study, fragments of \ac{exd} and \ac{ubx} containing their \acp{HD} and \acp{C-ter} were used. The reported binding sequences were ATCGAAATG and ATAAAACAA. \ac{exd} increased \ac{ubx} DNA binding by greatly reducing its dissociation rate. Surprisingly, a monoclonal antibody directed against the \ac{ubx} \ac{C-ter} stabilized this interaction as well, but \ac{antp} did not produce the same effect. However, when the interaction requirements were tested in the mammalian counterparts of \ac{ubx} and \ac{exd}, the resulting picture was quite different (\cite{Chang1995})[complete:4th alpha helix, hexapeptide. add \cite{Lu1996}.Move to interactions].

The vertebrate Hox \acp{HD} recognize a short DNA sequence of 4 nucleotides that in most cases contains a TAAT core (\cite{Treisman1992,Catron1993}[others, the Drosophila ones]). The nucleotides flanking the TAAT core affect in binding differentially according to the paralog group, but the magnitude of this effect is small. In addition, there is a gradient of DNA affinity such that 3' Hox \acp{HD} have higher DNA affinities than 5' ones (\cite{Pellerin1994}). [has this been borne by later studies? \cite{Shen1997a} says the homodimers have differing stabilities on DNA]

The Pbx1 \ac{HD} was shown to bind TGATTGAT (\cite{VanDijk1993}), but this could have been an artifact of the GST-\ac{HD} fusions used (\cite{review que lo decia: Mann and chan 1996?}. When Pbx-Hox heterodimer DNA binding was tested extensively, it was found that the the slight differences in DNA selectivity shown by individual Hox monomers are exacerbated and the preferred target sites shift for most of the complexes. The Pbx-Hox consensus target sequence core is TGATNNAT, where the 5' TGAT part is bound by Pbx and the 3' NNAT is bound by the Hox protein. The Hox core nucleotides differ according to the Hox protein participating in the complex: 3' Hoxes show highest affinity for a TGAT half-site, middle Hoxes (paralog groups 4-7) bind a half site that matches the monomeric Hox site TAAT, and 5' Hoxes (groups 8 and higher) are very strict in their preference for a TTAT half site \textit{vis-Ã -vis} other NNAT variants, but are also able to bind TTAC (\cite{Chang1996,Chan1997,Shen1997a}). Summarized in Figure \ref{fig:Shen1997JBC_fig5}.

The \ac{aa} residues determining sequence specificity have been explored. One of the most important is Arginine 5 of the \ac{HD}, (\cite{Phelan1997})[Noyes2008 investigates aa sequence-recognition site correspondence in depth] 


\begin{figure}[]
  
  \centering
  \label{fig:Shen1997JBC_fig5}
  \includegraphics[width=0.5\textwidth]{figures/Shen1997JBC_fig5}
  \caption[Pbx-Hox heterodimers show different base preferences at position 7 of their target sequence]{\textbf{Pbx-Hox heterodimers show different base preferences at position 7 of their target sequence.} From \cite{Shen1997a}}
\end{figure}

Meis can bind DNA \textit{in vitro} in the absence of either Pbx or Hoxes. In this case it recognizes a very strict consensus sequence, TGACAG, often in multiple copies without a clear orientation or spacing (\cite{Shen1997}). This target sequence is very similar to that of the TGIF family of TALE proteins \cite{ref}. This binding is highly unstable despite being quite  intense in the steady state. Meis can also bind in complex with HoxA9, in which case the heterodimer recognizes a sequence containing TGACAG and the HoxA9 targets TTAT or TTAC in a fixed spacing (\cite{Shen1997}). When Meis binds DNA in a complex with Pbx, the preferred recognition target is the decameric sequence TGATTGACAG, containing a 5' TGAT target and a 3' TGACAG Meis target (\cite{Chang1997}). Several Pbx-Hox target sites have been tested for binding to Pbx-Meis, and the only ones to bind were those containing a G in position 5 of the Pbx-Hox target octamer. 

Prep can bind DNA as a monomer, recognizing the same hexameric TGACAG sequence as Meis (\cite{Berthelsen1998a}). In this study, Pbx1 interaction was shown to increase DNA binding but no change in specificity was tested.

Prep can also bind DNA as an heterodimer with Pbx, in which case can recognizes either the TGACAG monomer target or a Pbx-Hox canonical sequence, TGATTGAT or TGATGGAT (\cite{Berthelsen1998}). 

% UEF3 has negative regulatory function in several of its target genes

[union de meis, prep, triples]

\begin{figure}[]
  
  \centering
  \label{fig:Shen1997MCB_fig9}
  \includegraphics[width=\textwidth]{figures/Shen1997MCB_fig9}
  \caption[Meis and Pbx interact with different Hox paralogs]{\textbf{Meis and Pbx interact with different Hox paralogs.} From \cite{Shen1997}}
\end{figure}

Immunoprecipitation of Pbx, Meis or HoxA9 complexes formed \textit{in vitro} found essentially the same TGATTTAT sequence regardless of the complex component targeted (\cite{Shen1999}). This is a consensus Pbx-Hox binding site that Meis1 on its own is not able to bind.

However, the above described consensus sequences represent the binding site for which a particular Pbx-Hox heterodimer has the most affinity. It may be that the important factor \textit{in vivo} is not the affinity but the selectivity of a particular site that allows it to choose between many Pbx-Hox heterodimers. An example is the Hoxb1 autoregulatory enhancer. It contains several Hox-Pbx binding sites, of which one, R3, is highly specific for HoxB1-Pbx1. In Drosophila, a similar sequence is present in a labial autoregulatory enhancer. Changing the sequence of this lab-\ac{exd} target to TGATTAAT switches the specifity to dfd-\ac{exd} (\cite{Chan1997}). However, the TGATGGAT original sequence is not the maximum affinity consensus for lab-\ac{exd} or any vertebrate Pbx-Hox heterodimer to begin with (\cite{Mann1998,Shen1997}). The possibility that selectivity is more important than affinity is borne by the fact that many \acp{HD}, despite having identical highest-affinity targets, differ in their "secondary" target motifs (\cite{Berger2008}).

If this is true, then \textit{in vitro} binding studies that consider individual Hox proteins or at most combinations of a single Hox protein with different cofactors are severely limited in their potential usefulness. Despite being informative about the physical properties of Hox DNA binding, [competition for sites, etc]
		
Several relatively recent studies have applied high throughput techniques to the description of \ac{HD} binding preferences. (\cite{Slattery2011,Berger2008,Noyes2008})
		
TGATGGAT: \cite{Poepperl1995,Chan1997} (in vivo). \cite{Chang1997} (in vitro)

TGATTGACAG: \cite{Knoepfler1997}

[TGACAG, others]

[\textit{"(...) indicate that \ac{HD} proteins display different degrees of specificity when different experimental systems are used. It seems unlikely that the sequence specificities observed in DNA binding studies in vitro or even cultured cell experiments are sufficient to account for the specificity of the proteinsâ actions in living flies."} (\cite{Hayashi1990}).]

[\textit{"This shows that functional specificity in vivo is a more subtle and complex property than has been addressed by studies of DNA binding specificity in vitro"} (\cite{Treisman1992}).]



\section{Expression Patterns}

[ISH de Prep?]


\section{Nuclear Localization of TALE proteins}
\label{sec:nuclearLoc}

Pbx/exd is present in the cytoplasm in the absence of Meis/hth and nuclear in its presence. This is due to the presence in it of at least a \ac{NES} and a \ac{NLS}. The \ac{NES} and \ac{NLS} have been mapped to \acp{aa} 178-220 and 239-243 of exd, respectively (\cite{Abu-Shaar1999}). These positions fall within the PBC-B motif and the \ac{N-ter} of the \ac{HD}. Meis/hth has a \ac{NLS} that allows it to target \ac{NLS}-deficient Pbx/exd fragments to the nucleus (\cite{Abu-Shaar1999}). Other reports have described an additional \ac{NES} within PBC-A (\cite{Berthelsen1999}) and a \ac{NLS} within the 3\textsuperscript{rd} helix of the \ac{HD} (\cite{Saleh2000}). The nuclear/cytoplasmic balance can be altered by leptomycin B, a nuclear export inhibitor, suggesting the role of Meis/hth is to retain Pbx/exd in the nucleus (\cite{Berthelsen1999, Abu-Shaar1999}). Two somewhat overlapping models for nuclear import/export of Pbx/exd have been proposed. The first considers a balance between \acp{NES} and \acp{NLS} which gets tipped by binding to \ac{NLS}-containing Meis/hth or Prep (\cite{Affolter1999}). The second, supported by point mutations of Pbx, implies intramolecular interactions within Pbx/exd between its \ac{N-ter} and \ac{HD} that mask the \acp{NLS} and are disrupted by binding of Meis/hth or Prep to its \ac{N-ter} (\cite{Saleh2000}).

Another regulator of Pbx subcellular localization is nonmuscle myosin heavy chain (\cite{Huang2003}). Pbx binds it through its PBC-B domain and zipper =the nonmuscle myosin heavy chain Drosophila homolog) show increased cytoplasmic retention of \ac{exd}. When overexpressed, zipper can even induce cytoplasmic localization of \ac{hth}, presumably through \ac{exd}

Prep also requires co-expression with Pbx/exd for its nuclear localization (\cite{Berthelsen1999}). 



\section{Functions and phenotypes}

		Meis1  \cite{Azcoitia2005, Carramolino2010}. [fill] [Other Meises - ask Laura for summary]. Joint overexpression of Meis1 and Hoxa7 or Hoxa9 is sufficient to induce myeloid leukemia (\cite{Nakamura1996}).
		Prep1 null, \ac{prepi} \cite{Ferretti2006, Fernandez-Diaz2010, Longobardi2010}. [fill] [Other Preps?]
		Pbx1 deletion results in both hematopoietic \cite{DiMartino2001} and patterning \cite{Selleri et al. 2001} defects in mice.
		
\subsection{Knock-Out Phenotypes}

\subsection{Involvement in Leukemias}


\section{Regulation of gene expression}
[meter o no meter?]
\subsection{Promoters, RNApolII, TFS}

\subsection{Enhancers and enhancer-binding proteins}

\subsection{Epigenetics: histone marks}

\subsection{Chromatin conformation}

\section{Intro to ChIP-seq}

\subsection{Original description}
The \ac{chip} assay consists of cross-linking of chromatin to preserve the non-covalent interactions between proteins and DNA for biochemical assay. It was first described by \cite{Solomon1988}. For many years its application remained confined to focused application \cite{Mardis2007}.  

The first genome-wide modification of the technique involved microarray assaying of the enriched DNA fragments (ChIP-chip, \cite{Ren2000}). As massive sequencing has gone down in cost, its application to assaying ChIP-enriched DNA fragments has increased. ChIP-seq \cite{Robertson2007} [complete]

[Comparison between chipchip and chipseq: el paper del chipexo]

[Ubx ChIP 1990: \cite{Gould1990}]

\subsection{methodology, shortcomings}




[meter Deep homology (eg tinman-nkx2.5, hth-Meis, Pax6-eyeless)?]
