
A) Intro to TALEs and Hoxes
	1. Hoxes

		1.1 history, Function
		William Bateson defined the term homeosis in 1894 to describe morphological variations in which a part of the body is substituted for another. Some of the genes resposible for these homeotic mutations [must therefore be responsible] for the identity of body parts. Many such mutations were discovered during the era of classical genetics in Drosophila. Deletion of Ultrabithorax, for example, results in a shift of the identity of the third thoracic segment into second thoracic, as shown by the presence of wings instead of halteres \cite{ref}.  
		1.2 structure, DNA binding, regulation of Hoxes
		In [when?] it was discovered that Drosophila homeotic genes are organized in two clusters, the Antennapedia and Bithorax complexes \cite{Lewis1978, Kaufman1990}. 
		All Hox genes share a 180 bp sequence, the homeobox. It codes for a 60aa DNA binding domain, the homeodomain. It is formed by three alpha-helices, of which [structure, DNA contact description].
		1.3 Downstream: the need for cofactors
		Hox genes are transcription factors. They need to coordinate the expression of many targets (few of which are known, though), activated or repressed differentially by different paralogs, despite their very short recognition sequence. This contradiction baffled biologists for [how long? who? not sure about the word...], but the discovery of DNA binding partners attenuated the [controversy no encaja, intensity of the question?]. The Hox cofactors [fill]


	2. history and general perspective of Hox cofactors

	Pbx was identified in Drosophila in 1990 as extradenticle \cite{Peifer1990} and proposed as a Hox protein cofactor based on phenotype. In parallel, it was identified as the DNA binding part of the chimeric protein produced by the t(1;19) translocation found in human pre-B cell acute lymphoblastic leukemia \cite{Kamps1990}. 
	Meis1 was identified in 1995 as a proto-oncogene locus in which the ecotropic murine leukemia virus {check} is inserted in BXH-2 mice \cite{Moskow1995}. Homothorax was identified in \cite{Rauskolb1995,Rieckhof1997} as a Drosophila homolog of Meis1 required for Exd nuclear localization. 
	Prep1 was identified in 1998 as a component of the urokinase enhancer factor 3 capable of forming a DNA binding-independent heterodimer with Pbx1 \cite{Berthelsen1998}.

	3. philogeny: hoxes, tales

		The homeodomain is present in many genes other than Hoxes. Homologs of homodomain genes have been found as far from vertebrates as plants [Knotted, cual mas?, buscar algo: mirar \cite{Burglin1997}]. Meis, Prep and Pbx are part of a subgroup of homeodomain proteins characterized by the presence in their homeodomain of a Three Aminoacid Loop Extension (TALE) that gives name to the class and is situated between alpha-helices 1 and 2 [recheck] \cite{Burglin1994, Mukherjee2007, Moens2006}

	4. structure of the TALE genes and proteins
	
		There are six distinct subgroups of TALE homodomain proteins in bilaterian genomes. Of these, only 3 are known to be Hox cofactors [verify]. These three groups, the PREP, MEIS, and PBC classes, share homologous sequences at their N-terminal ends \cite{BÃ¼rglin1998}. These sequences, called HM-1 and HM-2 in the MEIS and PREP classes and PBC-A and PBC-B in the PBC class, are exclusive to the three Hox cofactor classes despite their being apart in the TALE superclass philogeny \cite{Mukherjee2007}.
		The PBC class is the most divergent within the TALE superclass

	5. interactions

		Pbx was the first of the three cofactors to be shown to bind DNA cooperatively with Hoxes \cite{ref}. Over the years, a number of protein complexes involving two or more partners binding on the surface of DNA or independent of it have been described in vitro and [in vivo]. The picture emerging from two decades of in vitro studies is complex. 
		All three classes of TALE Hox cofactors have the nominal [is this right usage?] extension in their homeodomains, but only Meis and Pbx proteins have been shown to bind Hoxes directly [verify Prep hasn't]. Pbx is described in the literature as being the main Hox cofactor \cite{ref}. It binds Hox proteins through its TALE and the Hox YPWM motif, present in paralog groups 1 to 10 \cite{refs, Passner1999}. Meis can bind directly Hoxes belonging to groups 9 to 13 through its carboxy-terminal domain [what part of the hoxes?] \cite{ref}. 
		Both Meis and Prep are capable of forming heterodimers with Pbx proteins \cite{ref}. In the case of Meis/hth, this binding is required for the nuclear localization of the Pbx/exd protein {ref}. 
		In addition to these heterodimeric complexes, a number of trimeric complexes have been reported. In these, generally one of the protein partners does not bind DNA. [expand]

		[other interactions - PKA etc]

	6. binding specifities
		
		[TGAT, NNAT, TGACAG, TGATTGACAG, TGATNNAT, others]

	7. Functions, phenotypes:

		Meis1 \cite{Azcoitia2005, Carramolino2010}. [fill] [Other Meises?]. Joint overexpression of Meis1 and Hoa7 or Hoxa9 is sufficient to induce myeloid leukemia {ref}.
		Prep1 null, Prep1$^i/i$ \cite{Ferretti2006, Fernandez-Diaz2010, Longobardi2010}. [fill] [Other Preps?]
		Pbx1 deletion results in both hematopoietic \cite{DiMartino2001} and patterning \cite{Selleri et al. 2001} defects in mice.

B) Regulation of gene expression

	1. Promoters, RNApolII, TFS

	2. Enhancers and enhancer-binding proteins

	3. Epigenetics: histone marks

	4. Chromatin conformation

C) Intro to ChIP-seq

	1. original description
		The Chromatin Immunoprecipitation (ChIP) assay consists of cross-linking of chromatin to preserve the non-covalent interactions between proteins and DNA for biochemical assay. It was first described by \cite{Solomon1988}. For many years its application remained confined to focused application \cite{Mardis2007}.  
		The first genome-wide modification of the technique involved microarray assaying of the enriched DNA fragments (ChIP-chip, \cite{Ren2000}). As massive sequencing has gone down in cost, its application to assaying ChIP-enriched DNA fragments has increased. ChIP-seq \cite{Robertson2007}
	2. methodology, shortcomings




