\chapter{Introduction}
\label{chp:intro}
\thispagestyle{cleared}
\cleardoublepage

Developmental Biology deals with the process by which a single cell, the zygote, gives rise to trillions of specialized cells belonging to many different cell types that function as a coordinated whole.

In the process from zygote to adult, that single original cell must proliferate to spawn countless descendants. These descendants must also differentiate appropriately according to their position in the body. 

The key to differentiation are transcription factors, proteins that bind DNA and regulate the expression of nearby genes. By restricting active genes to the specific repertoire required to differentiate a particular cell type, they are responsible for that cell's identity. 

A very special class of transcription factors are responsible for translating and refining early, rough regions in the embryo into a system of positional information, instructing later cell fate decisions. They are the Hox genes, and they are present in all bilaterian animals with a similar genomic organization in most of them, homologous roles in patterning and extreme sequence conservation.%[instruct]

\section{The Hox genes}

\subsection{History of Hox Genes Study}

William Bateson defined the term homeosis in 1894 to describe morphological variations in which a part of the body resembles another. Homeotic mutations thus affect genes important for establishing body part identity. Several such mutations were discovered during the era of classical genetics in Drosophila. Deletion of \ac{ubx}, for example, results in a shift of the identity of the third thoracic segment into second thoracic, as shown by the presence of wings instead of halteres \parencite{Morata1981}.

%[Selector gene hypothesis (\cite{Garcia-Bellido1975,Garcia-Bellido1977}) include? fill]

After the work of many developmental geneticists culminating in Edward B. Lewis' landmark paper \parencite{Lewis1978} the basic rules of segmental specification in Drosophila were extensively understood. There were a number of mutations known to result in anterior to posterior transformations. It was known that these mutations involved two clusters of linked genes, the \ac{ANT-C} and the \ac{BX-C}. It was known that loss of function mutations produce anteriorization of segmental identity while gains of function result in posteriorization. %[kaufman1990:opposite for \ac{ANT-C} (?) \ac{ANT-C} when?]

Analysis of the phenotypes and their combinations indicated that there are two corresponding gradients in the embryo: a proximo-distal gradient in the chromosome and a \ac{BX-C} gene action antero-posterior gradient such that any gene active in a segment is active in all segments posterior to it--- these studies were performed before \textit{in situ} hybridization was developed or even the actual products of the genes affected were known.

%[Polycomb was known to be a repressor of all genes in the cluster necessary for proper patterning. It was known that segmental identity is cell-autonomously established. ]

The work of Christiane Nüsslein-Volhard and Eric F. Wieschaus expanded in this solid foundation \parencite{Nuesslein-Volhard1980}. They performed an extensive screen that identified many of the genes that would occupy developmental biologists for more than a decade. %[eliminate? not really relevant for Hoxes. If not, complete]

After the cloning of the Drosophila \ac{BX-C} \parencite{Bender1983} and \ac{ANT-C} \parencite{Garber1983,Scott1983}, it was noted that probes for \ac{antp} cross-hibridized with several other genes contained in them, indicating sequence similarity \parencite{McGinnis1984}.  The stretch of homology spans 180 base pairs, was called the homeobox and codes for a protein domain called the \ac{HD}. The \ac{HD} is even more conserved in sequence than the homeobox and from the very beginning was suspected to be responsible for DNA binding due to its high content in basic aminoacids  \parencite{Gehring1985}.

Understanding of the homeotic selector genes and of animal development in general underwent a revolution when, shortly afterwards, it was discovered that the homeobox is present in vertebrates. In two papers published back to back, Carrasco and co-workers reported cloning of a Xenopus gene with homology to \ac{antp}, \ac{ubx} and \ac{ftz} \parencite{Carrasco1984} while McGinnis and co-workers reported the sequence of these three Drosophila genes and, strikingly, the presence of homologs in several animal species far removed philogenetically from insects \parencite{McGinnis1984b}. In a context where no vertebrate genes controlling development were known, this must have been an earth-shattering revelation. Suddenly a system well studied in Drosophila potentially played a role in shaping the animals closest to us. It hinted at a deep underlying layer of similarity uniting all complex animal body plans that had been unsuspected before, and that would be confirmed and explored in the years to come.

%The various described genetic functions were grouped into three complementation groups: Ubx, Abd-A and Abd-B (\cite{Sanchez-Herrero1985}). %[is this the actual description of abd-A?si:kaufman1990. Irrelevant?]

%[Colinearity term earliest use I found:\cite{Lewis1985}]

By the early 1990s, It was well established that the Hox clusters of vertebrates and the \ac{HOM-C} of Drosophila were homologous. The collinearity of gene expression domains with respect to position along the chromosome was long since described in  Drosophila, and it had been recently reported that in vertebrates there is additional collinearity with timing of gene expression and RA sensitivity. Hox proteins were known to be able to bind DNA and suspected to be transcription factors (reviewed in \textcite{Levine1988, McGinnis1992}).

\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/HoxClusters}
  \caption[The Organization of Hox Genes]{\textbf{The Organization of Hox Genes.} In mammals there are four Hox clusters, containing a total of 39 genes arranged in the same orientation in each cluster. The Hox clusters of mammals are fairly compact and devoid of non-Hox genes. Here the murine clusters are shown to scale.}
  \label{fig:hoxclusters}
\end{figure}

A new stage in the study of Hox genes started focusing on their molecular function, through the use of homologous recombination in the mouse to generate loss of function and \textit{in vitro} techniques to characterize protein activity. The recent understanding of the high conservation of Hox gene function and organization across most of the animal kingdom would aid by making findings in different model organisms directly translatable. 

%[Ubx ChIP 1990: \cite{Gould1990}]
%[first mouse knockout \cite{Thomas1987}]

%[who and when proved molecular function of hoxes as TFs? Levine? in Levine1988 it is suggested, in McGinnis1992 it is established]

%[who and when developed ISH?]

%\ac{ANT-C}: reviewed in \cite{Kaufman1990}

\subsection{Cluster and Gene Structure}

Drosophila homeotic genes are organized in two genetic clusters, the  Bithorax (reviewed in \cite{Lewis1978}) and \ac{ANT-C} (reviewed in \cite{Kaufman1990}) complexes. The two of them form the \ac{HOM-C}. The Drosophila "cluster" was the first to be described, but in many aspects it is quite atypical. Both components are located in the third chromosome of the fly but separated by about 10Mb, more than one third of the total length of the chromosome. They are much longer than their vertebrate counterparts. Several non-Hox-related genes are contained in the \ac{ANT-C} and not all the Hox genes share the same transcriptional orientation. 

In mammals there are 39 Hox genes grouped in four tightly organized clusters, each of which is homologous to the Drosophila \ac{HOM-C}. There are 13 different sets of homologous genes (paralog groups) but no cluster contains examples of all paralogs. The only paralog groups present in all clusters are 4, 9 and 13, homologs of Drosophila Deformed (4) and Abdominal B (9 and 13). 

%[for figure footnote: The 5' part of the clusters is clearly different evolutionarily: all are homologs to AbdB, etc[complete], particular pattern of interactions, conservation of 4 paralogs in 9 and 13, ] 

For a more extensive comparative analysis of Hox clusters see \textcite{Duboule2007}.

All Hox genes share a 180 bp sequence, the homeobox. The homeobox codes for a 60aa DNA binding domain, the \ac{HD}, related to the helix-turn-helix DNA-binding domain of prokaryotes. It is constituted by a flexible \ac{N-ter} arm and three alpha-helices, of which the third slots into the major groove of DNA and interacts with specific nucleotide residues. The specificity of this interaction is critically affected by \ac{aa} 9 of the recognition helix \parencite{Treisman1992}. \acp{aa} 6 and 10 also contact DNA and are very conserved across \acp{HD}. Arginines at positions 3 and 5 of the \ac{N-ter} non-helical portion of the \ac{HD} also contact DNA on the core recognition site, but in the the minor groove side. Arginine 5 is almost invariant across \acp{HD}. Arginine 3 is present in Hox groups 2-8 and contributes to a preference for TAAT sites. The presence of a Lysine in position 3 is characteristic of the \ac{Abd-B} subfamily and probably contributes to its preference for TTAT (see below).

The hexapeptide/linker/\ac{N-ter} of the \ac{HD} is present in all Hox non-Abd-like proteins (paralogs 1-8). Paralogs 9 and 10 have an alternative tryptophan-containing motif \ac{N-ter} of the \ac{HD}. These motifs are relevant for Hox-cofactor interactions (see below).

Transcriptional activator/repressor functions are mediated by domains other than the \ac{HD} and differ between paralog groups \cite{ref}. The balance between activation and repression is also influenced by \ac{PKA} signalling \parencite{Saleh2000a}.

\subsection{Regulation of the Hox Genes}

Even before the discovery of the mouse Hox genes, it was already known that Hoxes have complex patterns of auto- and cross-regulation. 

%[auto- and cross- regulation]

Polycomb was described as early as 1978 as a repressor needed for the correct expression of Hox genes \parencite{Lewis1978}. In addition, the trithorax complex is an epigenetic activator of gene expression that also plays a role in the proper expression of Hox genes.%[complete]

The first diffusible molecule to be described as a regulator of Hox gene expression was \ac{RA} \parencite{Simeone1990}, which shifts their expression domains anteriorly. \ac{RA} signalling is mediated by retinoid acid receptors and retinoic X receptors. These are cytoplasmic proteins that translocate to the nucleus in the presence of \ac{RA} and activate gene expression by binding to \acp{RARE}. A number of \acp{RARE} have been described in the Hox clusters \cite{refs}.  \ac{RA} also upregulates Pbx levels through post-transcriptional means (\cite{Knoepfler1997a}) and Meis levels (\cite{ref}). %[expand Hox cluster RAREs]

Krox20 (\cite{Swiatek1993}) is a \ac{TF} that plays an important part in establishing Hox pattern in the rhombomeres of the vertebrate neural tube. %[fill]

Apart from Krox20, other regulators of Hox gene expression in the rhombomeres are Kreisler (\cite{ref}), Cdx1/2 (\cite{refs}), AP-2 (\cite{refs}), GATA-1 (\cite{ref}), %[others?]

%[RA induces Meis but not Prep, "ternary complex formation is induced by RA in embryonal carcinoma P19 cell" (Ferretti2000)]

%[miRNAs]


\subsection{Downstream: the Need for Cofactors}
%[DNA binding]

An important aspect of the effect of Hox genes in development is called phenotypic suppression \parencite{Gonzalez-Reyes1990} or posterior prevalence \parencite{Lufkin1991} and consists on the ability of more posterior (i.e. 5') Hoxes to suppress the effect of more 3' Hoxes. It explains the tendency of loss of function mutations to result in anteriorization while gain of function mutations usually result in posteriorization.

Hox genes are \acp{TF}. A simple view of their molecular effect is that they bind DNA and activate or repress the transcription of their nearby genes. However, all of the Hox \acp{HD} recognize slightly different  variants of the same, very short, DNA sequence. 

They need to coordinate the expression of many targets, few of which are known, that are activated or repressed differentially by different paralogs, despite their very short recognition sequence. This contradiction baffled biologists for years until the discovery of DNA binding partners attenuated the contradiction (see \cite{Mann1996}). The Hox cofactors enhance the specifity and affinity of Hox DNA-binding, thus providing a better explanation for functional selectivity than DNA sequence recognition by Hox proteins alone can provide. We will see however that the question is still far from resolved. 

%[Hox protein regulation of translation: look for the paper].

\section{History and general perspective of Hox cofactors}

Pbx was identified in Drosophila in 1990 as \ac{exd} (\cite{Peifer1990}) and proposed as a Hox protein cofactor based on phenotype. In parallel, it was identified as the DNA binding part of the chimeric protein produced by the t(1;19) translocation found in human pre-B cell acute lymphoblastic leukemia (\cite{Kamps1990}). They were recognised as homologs in 1993 (\cite{Rauskolb1993}).

Meis1 (Myeloid ecotropic insertion viral site 1) was identified in 1995 as a proto-oncogene locus in which the ecotropic murine leukemia virus is inserted in BXH-2 mice (\cite{Moskow1995}). \ac{hth} was identified as a Drosophila homolog of Meis1 required for \ac{exd} nuclear localization (\cite{Rauskolb1995, Rieckhof1997}).

Prep1 (Pbx-regulating-protein 1) was identified in 1998 as a component of the urokinase enhancer factor 3 capable of forming a DNA binding-independent heterodimer with Pbx1 \parencite{Berthelsen1998,Berthelsen1998a}.

\subsection{Philogeny: hoxes, tales}

The \ac{HD} is present in many genes other than Hoxes. Homologs of homodomain genes have been found as far from vertebrates as plants. Meis, Prep and Pbx are part of a subgroup of \ac{HD} proteins characterized by the presence in their \ac{HD} of a \ac{TALE} that gives name to the class and is situated between alpha helices 1 and 2 \parencite{Bertolino1995, Burglin1997, Mukherjee2007, Moens2006}. 

%[Knotted, Yeast mating proteins, others, buscar: mirar \cite{Burglin1997}]
\subsection{Structure of the TALE genes and proteins}

There are six distinct subgroups of \ac{TALE} homodomain proteins in bilaterian genomes. Of these, only 3 are known to be Hox cofactors. %[verify]

These three groups, the PREP, MEIS, and PBC classes, share homologous sequences at their \ac{N-ter} ends (\cite{Burglin1998}). These sequences, called HM-1 and HM-2 in the MEIS and PREP classes and PBC-A and PBC-B in the PBC class, are exclusive to the three Hox cofactor classes despite their being apart in the \ac{TALE} superclass philogeny (\cite{Mukherjee2007}) and are responsible for much of their Hox- and cross-interactivity (see below).

Several \acp{NES} and \acp{NLS} have been reported in Pbx and in Prep (summarized in Figure \ref{fig:TALE_Hox_protStructure}).

\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/TALE_Hox_protStructure}
  \caption[Structural domains of the TALE and Hox proteins]{\textbf{Structural domains of the TALE and Hox proteins.}}
  \label{fig:TALE_Hox_protStructure}
\end{figure}

The PBC class is the most divergent within the \ac{TALE} superclass. Members of this class have a fourth alpha helix 3' to the \ac{HD} \parencite{Mukherjee2007}. 

Residue 50 of the \ac{HD}, the 9\textsuperscript{th} of the 3\textsuperscript{rd} helix, is important for base selection by contacting the major groove of DNA and is a glicine in the PBC class, in contrast to the long side chain \acp{aa} in Hox proteins. %[other cofactors?]

\subsubsection{Splicing Isoforms}

Meis1 and Meis2 are spliced to produce two major isoform families in vertebrates. These isoforms differ in their \ac{C-ter} portion. Meis1a, Meis2a and Meis2b are all short-\ac{C-ter} isoforms, while Meis1b, Meis2c and Meis2d encode a long-\ac{C-ter} isoform. Meis3 is similarly spliced. Both kinds of isoforms are responsive to \ac{TSA} and \ac{PKA} but the minoritary (under 10\% in vertebrates) long-\ac{C-ter} versions are much more potent transcriptional activators, both constitutively and in response to \ac{TSA} and \ac{PKA} \parencite{Huang2005, Irimia2011}. DNA-binding and TALE/Hox interactions (see below) of Meis are generally unaffected by splicing, since the \ac{HD} and \ac{N-ter} are constitutively spliced in. There are minoritary \ac{HD}-less isoforms in vertebrates and insects \parencite{Irimia2011}. 

The existence of splicing isoforms has also been reported for Prep. There are at least two isoforms of Prep2, one of which contains the full \ac{N-ter} including HM1 and HM2 but not the \ac{HD}, while the other is the full-length protein \parencite{Haller2004}.

\section{Interactions Between the Hox genes and their TALE cofactors}
\label{sec:interactions}

Pbx was the first of the three cofactor classes to be shown to bind DNA cooperatively with Hoxes \parencite{Chan1994}. Over the years, a number of protein complexes involving two or more partners binding on the surface of DNA or independent of it have been described \textit{in vitro} and \textit{in vivo}. The picture emerging from two decades of \textit{in vitro} studies is complex. 

\subsection{Two-way Interactions}

All three classes of \ac{TALE} Hox cofactors have the nominal extension in their \acp{HD}, but only Meis and Pbx proteins have been shown to cooperatively bind DNA with Hoxes. Pbx is described in the literature as being the main Hox cofactor \cite{ref}. Its binding to Hox proteins requires its \ac{HD} plus the following 15 \acp{aa} and the Hox hexapeptide/linker/\ac{N-ter} of the \ac{HD}, which is an important determinant of DNA specificity and ability to interact with Pbx, as shown by chimeric \ac{HD} experiments \parencite{Chang1996, Phelan1997}. The hexapeptide, also known as YPWM motif, is present in paralog groups 1 to 8 \parencite{Chang1995, Passner1999}. Paralog groups 9 and 10 can interact with Pbx through an alternative ANW motif located just 5' to their \ac{HD} \parencite{Chang1996, Shen1997a}. Paralog groups 11 to 13 can not bind Pbx1a despite most of them containing tryptophan residues 5' to their \ac{HD}. 

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/PbxHoxCrystal_hiRes_midgain}
  \caption[Hox-Pbx-DNA Complex structure]{\textbf{Hox-Pbx-DNA Complex structure.} X-ray crystallographic determination of the structure of a Pbx1-HoxA9-DNA complex, from \cite{LaRonde-LeBlanc2003}. The Hox protein is labelled in blue and Pbx in orange, with their hexapeptide tryptophan and \ac{TALE} shown as spheres. The 3\textsuperscript{rd} alpha-helices contact DNA directly in head-to-tail orientation. The conserved tryptophan slots into a hydrophobic pocket in Pbx partially formed by the \ac{TALE}. The 4\textsuperscript{th} alpha-helix in the divergent Pbx \ac{HD} can be seen at the extreme right. The \ac{N-ter} linker in HoxA9 is ordered and contacts the minor groove of DNA, in contrast to those in the other Hox proteins crystallized, HoxB1 and Ubx. The portions of Hox \ac{N-ter} to the conserved tryptophan motif and of the Pbx \ac{N-ter} to the \ac{HD} were not crystallized.}
  \label{fig:PbxHoxCrystal}
\end{figure}

Crystallographic structure determination of a Exd-Ubx-DNA complex (\cite{Passner1999}) and a Pbx1-HoxB1-DNA complex (\cite{Piper1999}) showed that Pbx and the Hox component of the heterodimer bind DNA on opposite sides of the helix. The \ac{N-ter} linker (20 \acp{aa} long in HoxB1 and 8 \acp{aa} long in Ubx) of the Hox protein extends over the DNA so that the hexapeptide contacts Pbx directly, in an hydrophobic pocket partially formed by the DNA recognition helix in the Pbx \ac{HD} and by its \ac{TALE}. There is also potential for the \ac{N-ter} of the recognition helix of Pbx to interact with the \ac{C-ter} of the Hox recognition helix, since the two helices are oriented head-to-tail and their ends are close. Later determination of an \ac{Abd-B} homolog, HoxA9, in complex with Pbx1 and DNA showed very similar arrangement of the proteins in the DNA \parencite{LaRonde-LeBlanc2003}. Despite HoxA9's hexapeptide having a radically diferent backbone structure from Ubx's or HoxB1's, the critical tryptophan residue slots into Pbx1's hydrophobic pocket in the exact same position. One of the few differences between the determined structures is that HoxA9's \ac{HD} N-terminal linker is ordered, unlike Ubx's or HoxB1's.

%[two more structures: in \cite{Joshi2007} Scr-Exd bound to specific Scr target or consensus target]

Meis1 can bind directly Hoxes belonging to groups 9 to 13 through its \ac{C-ter} portion. This interaction greatly stabilizes Meis DNA binding, which may occur as homodimers in the absence of Hoxes or Pbx. The homeodomain of Hoxes is necessary for binding to Meis, but further \acp{aa} \ac{N-ter} to the Hox \ac{HD} are necessary to stabilize the binding, at least in the case of HoxA9. The part of Meis1 required for Hox binding is located somewhere within the \ac{HD} or the 38 following \acp{aa}, which are common to Meis1a and Meis1b. The two main Meis isoform types thus do not seem to differ in their protein or DNA binding characteristics \parencite{Shen1997, Williams2005}. %[Is hth restricted to Ubx binding???]

Pbx and Meis can interact independently of DNA, and \textit{in vivo} immunopurified Pbx-containing complexes avidly bind the joint Pbx-Meis binding target (see \ref{sec:specificites}), suggesting that most of the Pbx in the cell is complexed with Meis or Prep \parencite{Knoepfler1997, Chang1997}. This interaction is required for Pbx/exd to translocate to the nucleus (\cite{Rieckhof1997}) and for stability of the Meis/\ac{hth} protein (\cite{Abu-Shaar1998}). The Pbx Meis-interacting surface is contained within its first 88 \acp{aa}, \ac{N-ter} to the point of fusion in E2a-Pbx1 chimeras and within the PBC-A domain. The Meis Pbx-interacting surface is located between \acp{aa} 30 and 60 \parencite{Chang1997}. The fact that Meis and Pbx interact through their flexible \ac{N-ter} arms results in non-strict position and orientation requirements for their DNA targets, according to \textcite{Jacobs1999}.

Prep binds Pbx independently of DNA through its \ac{N-ter} conserved domains and the Pbx part deleted in E2a-Pbx \parencite{Berthelsen1998}, like Meis. 

%see \cite{Mann1996}

\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/TALE_Hox_interactions}
  \caption[Summary of TALE-Hox interactions]{\textbf{Summary of TALE-Hox interactions.}}
  \label{fig:TALE_Hox_interactions}
\end{figure}


\subsection{Trimeric Complexes}

In addition to the heterodimeric complexes, a number of trimeric complexes have been reported. In these, generally one of the protein partners does not bind DNA. 

Meis and Pbx can form three-protein complexes with at least some of the Hox proteins. Meis1 can form ternary complexes with any of the Pbx's and HoxA9 \textit{in vitro} independently of DNA. Addition of Meis greatly enhances the  Pbx-HoxA9 interaction, which is weak in the absence of DNA (\cite{Shen1999}). In this study it was also shown that the three proteins colocalize \textit{in vivo} in nuclear speckles, but surprisingly fail to activate or repress a transcriptional reporter bearing their consensus DNA target. 

Meis has also been shown to form trimeric complexes with Pbx and HoxB1. These heterotrimers bound the HoxB1 ARE and the HoxB2 r4 enhancer sequences and required intact Pbx-Hox and Meis half-sites to bind DNA, but probably not to assemble (\cite{Jacobs1999}). Formation of complexes was checked to be dependent of the heterodimerization motifs of each protein component, implying that Pbx is the hinge subunit that interacts with both Meis and Hoxb1 at the same time. Addition of the three proteins resulted in cooperative transcriptional activation in a transient transfection reporter assay and \textit{in vivo} lacZ staining that depended on the motifs for Pbx-Hox and Meis binding, in contrast to \cite{Shen1999}. 

Trimeric Pbx-Meis-HoxA9 complexes are required for HoxA9-mediated immortalization of myeloid progenitors (\cite{Schnabel2000}).

% GGGGG sequence present between the OCTA and HEXA motifs in both r4enh and b1ARE

Prep can form triple complexes with Pbx and a Hox protein. Co-transfection of Prep1, Pbx1 and Hoxb1 resulted in additional activation of a Hoxb1 \ac{ARE} reporter over Pbx1 and Hoxb1 alone (\cite{Berthelsen1998}). This additional activation was independent of Prep1 DNA binding ability but required its ability to bind Pbx. Prep1-Pbx1-Hoxb1 trimeric binding has been directly demonstrated \textit{in vitro} on sequences from the rhombomeric Hoxb1 \ac{ARE} and on the Hoxb2 r4 enhancer (\cite{Ferretti2000}). In the case of the Hoxb1 \ac{ARE}, a non-DNA-binding version of Prep1 was surprisingly required for formation of the trimeric complex. %[MT:Esto también pasa con meis, ver paper de Featherstone]

\subsection{Interactions with Other Proteins}
%[other interactions - PKA etc]

Pbx-Meis/Prep heterodimers have been shown to be able to bind DNA in concert with E2a-Myogenin and E2a-MyoD heterodimers (myogenic \ac{BHLH} \acp{TF}) in the form of tetrameric complexes, as well as with myogenin alone. Hox proteins preclude this interaction, which requires a conserved tryptophan in the \ac{BHLH} \acp{TF} that is situated \ac{N-ter} to its DNA-binding domain and could thus be mediated by the same hydrophobic pocket in Pbx as Pbx-Hox interaction in a similar configuration (\cite{Knoepfler1999}). Pbx-Meis interaction with MyoD has been shown to be necessary for MyoD binding to the Myogenin promoter, which results in chromatin remodelling and activation of Myogenin, suggesting recruitment of MyoD to heterochromatin by Pbx-Meis (\cite{Berkes2004}).

% "...the fact  that deletion of the interaction motif on MyoD does not significantly alter synergy between MyoD and Pbx–Meis/Prep1 in co-transfection assays suggests that this interaction may be designed to juxtapose distant Pbx–Meis/Prep complexes next to bHLH–E2a complexes rather than to stabilize DNA binding..." (Knoepfler1999)

Pbx was shown to bind the pancreatic homeodomain protein PDX1 through a tryptophan motif similar to that of Pbx-interacting hoxes (\cite{Peers1995}). PDX1 can also form trimers with Pbx-Meis heterodimers. PDX1 is critical for pancreas formation and participates in the differentiation of islet and acinar pancreatic cells. Expression of Pbx1 and Meis2 and trimeric association with PDX1 is specific to acinar cells, representing a switch that changes the transcriptional effect of constitutive PDX1 binding at the ELA1 enhancer (\cite{Swift1998}). The Pbx-Meis-PDX1 trimer activates transcription with a \ac{BHLH} heteromultimer in the ELA1 enhancer independently of Meis DNA binding (\cite{Liu2001}).

The somatostatin promoter has been shown to be bound by Pbx1-Prep heterodimers but not Pbx-Meis. This binding is necessary for PDX1 to be able to activate somatostatin (\cite{Goudet1999}). It is interesting that in the somatostatin promoter, binding of Pbx-Prep heterodimers was able to induce reporters driven by isolated components of the regulatory sequence but not by the whole sequence, which required the addition of PDX1 for activation. 

Another non-Hox homeodomain protein, Engrailed, has also been shown to interact with Pbx through a conserved tryptophan \ac{N-ter} to its homeodomain, with an affinity comparable to Hoxes (\cite{Peltenburg1996}).

% kim et al 2002: genetic interaction pbx-pdx1

% otros hds con el motivo de interaccion: cdx1,2,4 (peers1995) cdx's no demostrados

% Pbx-Hand2. Hand2 is a bHLH protein like MyoD (Maves et al 2009) pero NO interacciona, sino que esta bajo el control de Pbx2/4


\section{DNA Binding Specifities}
\label{sec:specificites}
%[\cite{Levine1988} proposes in vitro transcription as a new method applicable to homeobox-containing genes. "it had been previously shown that fly and frog homeobox proteins bind to specific DNA sequence motifs"]
		
The first homeobox proteins to be shown to be sequence-specific \acp{TF} were \ac{ftz}, \ac{en} and \ac{eve} (\cite{Desplan1988,Hoey1988,Hoey1988a}). Two types of binding consensus sequence were described: TCAATTAAAT and TCAGCACCG.
		
\ac{ubx} \ac{CHIP} showed binding to sites containing a TAAT core (\cite{Gould1990}). This study also showed that binding to sites containing very similar sequences can result in upregulation or downregulation of transcripts, so the sign of transcriptional regulation is probably not encoded in the core target sequence.

%%%Binding site selection paper:
%Science. 1990 Nov 23;250(4984):1104-10.
%Differences and similarities in DNA-binding preferences of MyoD and E2A protein complexes revealed by binding site selection.
%Blackwell TK, Weintraub H.		

\ac{exd} was shown to bind cooperatively with \ac{ubx} to sequences in a \textit{dpp} enhancer \parencite{Chan1994}. In this study, fragments of \ac{exd} and \ac{ubx} containing their \acp{HD} and \acp{C-ter} were used. The reported binding sequences were ATCGAAATG and ATAAAACAA. \ac{exd} increased \ac{ubx} DNA binding by greatly reducing its dissociation rate. Surprisingly, a monoclonal antibody directed against the \ac{ubx} \ac{C-ter} stabilized this interaction as well, but \ac{antp} did not produce the same effect. However, when the interaction requirements were tested in the mammalian counterparts of \ac{ubx} and \ac{exd}, the resulting picture was quite different \parencite{Chang1995}.%[complete:4th alpha helix, hexapeptide. add \cite{Lu1996}.Move to interactions].

The vertebrate Hox \acp{HD} recognize a short DNA sequence of 4 nucleotides that in most cases contains a TAAT core \parencite{Treisman1992, Catron1993}. The nucleotides flanking the TAAT core affect in binding differentially according to the paralog group, but the magnitude of this effect is small. In addition, there is a gradient of DNA affinity such that 3' Hox \acp{HD} have higher DNA affinities than 5' ones \parencite{Pellerin1994}. %[has this been borne by later studies? \cite{Shen1997a} says the homodimers have differing stabilities on DNA]

%[others, the Drosophila ones]

The Pbx1 \ac{HD} was shown to bind TGATTGAT \parencite{VanDijk1993}, but this could have been an artifact of the GST-\ac{HD} fusions used. When Pbx-Hox heterodimer DNA binding was tested extensively, it was found that the the slight differences in DNA selectivity shown by Hox monomers are exacerbated when binding with Pbx and the preferred target sites shift for most of the complexes. The Pbx-Hox consensus target sequence core is TGATNNAT, where the 5' TGAT part is bound by Pbx and the 3' NNAT is bound by the Hox protein. The Hox core nucleotides differ according to the Hox protein participating in the complex: 3' Hoxes show highest affinity for a TGAT half-site, middle Hoxes (paralog groups 4-7) bind a half site that matches the monomeric Hox site TAAT, and 5' Hoxes (groups 8 and higher) are very strict in their preference for a TTAT half site \textit{vis-à-vis} other NNAT variants, but are also able to bind TTAC (\cite{Chang1996, Chan1997, Shen1997a}). %[\cite{review que decia lo del artefacto de GST? Mann and chan 1996?}

The TGATTGAT sequence has been shown to be bound by Pbx-Prep heterodimers in the context of the somatostatin promoter \parencite{Goudet1999}. The A/TGATAAAT sequence has been shown to be bound by trimeric Pbx-Meis-PDX1 complexes \parencite{Liu2001}.

The \ac{aa} residues determining sequence specificity have been explored. One of the most important is Arginine 5 of the \ac{HD}, which is conserved across all Hox proteins \parencite{Phelan1997}. %[Noyes2008 investigates aa sequence-recognition site correspondence in depth] 


\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/pbxHoxDNAbindingpreferences}
  \caption[Pbx-Hox heterodimer binding preferences]{\textbf{Pbx-Hox heterodimer binding preferences.} DNA binding preferences for each paralog group of Hox proteins The width of each bar at each position represents relative affinity of that paralog group for that sequence. In black, the three major Hox half-sites. In grey, the most important secondary binding targets. Thickness of bars indicates relative affinity. Adapted from \textcite{Shen1997a} and \textcite{Shen1997} to include data from \textcite{Slattery2011}.}
  \label{fig:PbxHoxBinding}
\end{figure}

Meis can bind DNA \textit{in vitro} in the absence of either Pbx or Hoxes. In this case it recognizes a very strict consensus sequence, TGACAG, often in multiple copies without a clear orientation or spacing \parencite{Shen1997}. This target sequence is very similar to that of the TGIF family of TALE proteins \cite{ref}. This binding is highly unstable despite being quite intense in the steady state, which implies a rapid turnover of the bound protein. 

Meis can also bind in complex with AbdB-like Hoxes (groups 9 to 13), in which case the heterodimer recognizes a sequence containing TGACAG and the AbdB-like Hox targets TTAT or TTAC in a fixed spacing \parencite{Shen1997}.

When Meis binds DNA in a complex with Pbx, the preferred recognition target is the decameric sequence TGATTGACAG, containing a 5' TGAT target and a 3' TGACAG Meis target \parencite{Chang1997, Knoepfler1997}. Gapped configurations in which the TGAT and TGACAG half sites are spaced by 3 or 6 nucleotides have also been shown to bind Pbx-Meis heterodimers \parencite{Jacobs1999}. Several Pbx-Hox target sites have been tested for binding to Pbx-Meis, and the only ones to bind were those containing a G in position 5 of the Pbx-Hox target octamer. %[ref?]. 

Prep can bind DNA as a monomer, recognizing the same hexameric TGACAG sequence as Meis \parencite{Berthelsen1998a}. In this study, Pbx1 interaction was shown to increase DNA binding but no change in specificity was tested.

Prep can also bind DNA as an heterodimer with Pbx, in which case can recognizes either the TGACAG monomer target or a Pbx-Hox canonical sequence, TGATTGAT or TGATGGAT (\cite{Berthelsen1998}).%[prep on DECA???]

% UEF3 has negative regulatory function in several of its target genes


%[trimeric complexes]

Immunoprecipitation of Pbx, Meis or HoxA9 complexes formed \textit{in vitro} found essentially the same TGATTTAT sequence regardless of the complex component targeted (\cite{Shen1999}). This is a consensus Pbx-Hox binding site that Meis1 on its own is not able to bind.

Ternary Meis-Pbx-Hox complexes are able to bind the Hoxb2 r4 enhancer, which contains a Pbx-Hox (AGATTGAT) and a Meis monomer target site (TGACAG), both of which are necessary for trimeric binding (\cite{Jacobs1999}). The Hoxb1 \ac{ARE} also contains a Pbx-Hox (TGATGGAT) and a Meis monomer target site (TGACAG) and is likewise capable of binding by the trimeric Meis-Pbx-Hox complex, requiring both sites for binding (\cite{Jacobs1999}).

Ternary Prep1-Pbx1-Hoxb1 complexes are able to target the same Hoxb1 r3 \ac{ARE} (TGATGGAT/TGACAG), but only if the Prep1 \ac{HD} or the Meis/Prep target site are eliminated. In contrast, they can bind the Hoxb2 r4 enhancer (AGATTGAT/TGACAG) with the endogenous forms of the proteins, and the elimination of the Prep1 \ac{HD} in this case results in reduction of DNA binding affinity (\cite{Ferretti2000}).

Tetrameric complexes involving Pbx, Meis or Prep, and a pair of myogenic \ac{BHLH} \acp{TF} can bind a Pbx-Meis/Prep target site (TGATTGACAG) in a minimum spacing of 3bp from an E-box (CAGCTG) (\cite{Knoepfler1999}).
	
Several relatively recent studies have applied high throughput techniques to the description of \ac{HD} binding preferences. Two such studies managed to predict \ac{HD} DNA binding preferences from \ac{aa} sequence alone by constructing mathematical models that used as input the affinity of a wide selection of \acp{HD}. One tested the affinity of \textit{in vitro} translated \acp{HD} for all possible 10-nucleotide combinations present in a protein binding microarray (\cite{Berger2008}). The second used an specifically designed bacterial one hybrid system to gather their starting data, affinity of \ac{HD} hybrids (\cite{Noyes2008}). Although an impressive technical feat, the relevance of the described monomeric binding affinities is doubtful especially in the case of Hoxes, which change their affinity and specifity when binding DNA with their cofactors. 

A later study overcame this limitation by restricting its test population to the 8 Drosophila Hox proteins and assessing their sequence preferences in complex with the Drosophila Pbx ortholog \ac{exd} and the HM domain of the Drosophila Meis ortholog \ac{hth}. \textcite{Slattery2011} combined the SELEX technique (\cite{Tuerk1990}) with massive sequencing and a sophisticated computational analysis to infer sequence to affinity relationships for all possible 12-mers relative to each Hox-cofactor complex. This study is not only the most comprehensive exploration of Hox-Pbx DNA binding, but also the \textit{in vitro} binding study that uses conditions most relevant to the understanding of \textit{in vivo} binding. 

\textcite{Slattery2011} reaffirmed the notion that Pbx-Hox complex specificity is limited to an 8-bp long core, in which the 5' Pbx-binding half is almost always TGAT and the 3' Hox-dependent half has three major variants (TGAT, TAAT, TTAT) recognized preferentially by different subsets of Hoxes, but uncovered fine details slightly divergent from the classical 1996-1997 studies:

\begin{itemize}

\item lab and pb (paralog groups 1-2) recognize mainly a TGAT half-site, but also bind with comparable affinities other octamers: GGAT and CGAT for lab, and a TGACTAAT site for pb. 

\item dfd and scr (groups 4-5) recognize TGAT, TAAT and TTAT with similar affinities, but are also able to bind a wide variety of secondary half-sites, the most characteristic of which is AAAT.

\item Antp, Ubx, abdA and abdB (groups 6-13) have a high relative affinity for TTAT, but are also able to bind the other two majoritary half-sites and a host of secondary sites, including one specific for them, TGATTGAC.  

\end{itemize}


% TGATCGAT: PH3 in r4 enhancer: tumpel et al 2007 - AGAT in PH2 \cite{Tuempel2007}
% TGATGGAT: \cite{Poepperl1995,Chan1997} (in vivo). \cite{Chang1997, Slattery2011} (in vitro)

% TGATTGACAG: \cite{Knoepfler1997}

% [TGACAG, others]

% [\textit{"(...) indicate that \ac{HD} proteins display different degrees of specificity when different experimental systems are used. It seems unlikely that the sequence specificities observed in DNA binding studies in vitro or even cultured cell experiments are sufficient to account for the specificity of the proteins’ actions in living flies."} (\cite{Hayashi1990}).]

% [\textit{"This shows that functional specificity in vivo is a more subtle and complex property than has been addressed by studies of DNA binding specificity in vitro"} (\cite{Treisman1992}).]

However, all these consensus sequences represent the binding site for which a particular Pbx-Hox heterodimer has the most affinity. It may be that the important factor \textit{in vivo} is not the affinity but the selectivity of a particular site that allows it to choose between many Pbx-Hox heterodimers. An example is the Hoxb1 autoregulatory enhancer. It contains several Hox-Pbx binding sites, of which one, R3, is highly specific for HoxB1-Pbx1. In Drosophila, a similar sequence is present in a \ac{lab} autoregulatory enhancer. Changing the sequence of this lab-\ac{exd} target to TGATTAAT switches the specifity to dfd-\ac{exd} (\cite{Chan1997}). However, the TGATGGAT original sequence is not the maximum affinity consensus for lab-\ac{exd} (\cite{Mann1998,Shen1997}). The possibility that selectivity is more important than affinity is borne by the fact that many \acp{HD}, despite having identical highest-affinity targets, differ in their "secondary" target motifs (\cite{Berger2008}).

If this is true, then \textit{in vitro} binding studies that consider individual Hox proteins or at most combinations of a single Hox protein with different cofactors may not represent the most relevant or the majority of functional interactions taking place \textit{in vivo}. 
%[competition for sites, etc]

\section{Expression Patterns}


Prep1 and Prep2 are ubiquitously expressed \parencite{Ferretti1999, Reymond2002, Ferretti2006}. 

Pbx1, Pbx2, and Pbx3 are widely expressed, while Pbx4 expression is restricted to both the embryonic and adult testes \parencite{Monica1991, Wagner2001}.

Meis1 and Meis2 expression is similar, starts around gastrulation and is regionalized \parencite{Cecconi1997, Oulad-Abdelghani1997, Tamplin2008}.

% [Meis1 ISH not found in the literature]


\section{Nuclear Localization of TALE proteins}
\label{sec:nuclearLoc}

Pbx/exd is present in the cytoplasm in the absence of Meis/hth and nuclear in its presence. This is due to the presence in it of at least a \ac{NES} and a \ac{NLS}. The \ac{NES} and \ac{NLS} have been mapped to \acp{aa} 178-220 and 239-243 of exd, respectively (\cite{Abu-Shaar1999}). These positions fall within the PBC-B motif and the \ac{N-ter} of the \ac{HD}. Meis/hth has a \ac{NLS} that allows it to target \ac{NLS}-deficient Pbx/exd fragments to the nucleus (\cite{Abu-Shaar1999}). Other reports have described an additional \ac{NES} within PBC-A (\cite{Berthelsen1999}) and a \ac{NLS} within the 3\textsuperscript{rd} helix of the \ac{HD} (\cite{Saleh2000}). The nuclear/cytoplasmic balance can be altered by leptomycin B, a nuclear export inhibitor, suggesting the role of Meis/hth is to retain Pbx/exd in the nucleus (\cite{Berthelsen1999, Abu-Shaar1999}). Two somewhat overlapping models for nuclear import/export of Pbx/exd have been proposed. The first considers a balance between \acp{NES} and \acp{NLS} which gets tipped by binding to \ac{NLS}-containing Meis/hth or Prep (\cite{Affolter1999}). The second, supported by point mutations of Pbx, implies intramolecular interactions within Pbx/exd between its \ac{N-ter} and \ac{HD} that mask the \acp{NLS} and are disrupted by binding of Meis/hth or Prep to its \ac{N-ter} (\cite{Saleh2000}).

Another regulator of Pbx subcellular localization is nonmuscle myosin heavy chain (\cite{Huang2003}). Pbx binds it through its PBC-B domain and zipper (the nonmuscle myosin heavy chain Drosophila homolog) mutants show increased cytoplasmic retention of \ac{exd}. When overexpressed, zipper can even induce cytoplasmic localization of \ac{hth}, presumably through \ac{exd}.

Prep also requires co-expression with Pbx/exd for its nuclear localization (\cite{Berthelsen1999}). Cytoplasmic PREP2 is associated with the actin and microtubule cytoskeleton components and this cytoplasmic localization is dependent on CRM-1-mediated nuclear export and the integrity of cytoskeletal networks (\cite{Haller2004}).


\section{Functions and phenotypes}

\subsection{Knock-Out Phenotypes}

Meis1 mutant mouse embryos die around E14.5 showing eye, angiogenesis and hematopoietic defects, among others (\cite{Hisa2004, Azcoitia2005}). Their pervasive haemorrhages result from incomplete separation of the blood and lymphatic vasculature systems, which in turn results from failure of the megakaryocytic lineage (\cite{Carramolino2010}). They also show a reduction in \ac{HSC} populations in fetal liver, and cardiac malformation. 

Prep1 \ac{KO} embryos die early in development at E7.5 due to massive epiblast p53-dependent apoptosis (\cite{Fernandez-Diaz2010}). To circumvent this obstacle to embryonic Prep function study, \ac{prepi} mice have been generated. \ac{prepi} embryos produce around 2\% of the \ac{WT} levels of protein and die at E17.5 with eye, angiogenesis and hematopoietic defects, apart from organ hypoplasia (\cite{Ferretti2006, DiRosa2007}).  

Although the phenotypic alterations in \ac{prepi} embryos affect the same processes altered in Meis1 \ac {KO} embryos, they have different effects. The vasculature defects are centered around the lymphatic valves in Meis1 \ac{KO} but around the capillaries in \ac{prepi}. The hematopoietic defects consist in total lack of the megakaryocytic lineage in Meis \ac{KO} but in  \ac{prepi} there is an incomplete maturation of erythrocytes. Those \ac{prepi} embryos that escape embryonic lethality show defects in T-cell development (\cite{Penkov2005}). 
%[Other Preps?]

Pbx1 deletion results in both hematopoietic (\cite{DiMartino2001}) and patterning (\cite{Selleri2001, Moens2006}) defects in mice. Many of these defects phenocopy individual Hox mutants, and combined knockdowns of Pbx1 and different Hoxes indicate genetic interaction (\cite{Moens2006}).

Pbx1/PDX1 double heterozygous mice have defects in pancreas formation and develop diabetes mellitus (\cite{Kim2002}). 

Zebrafish Pbx has been shown to be necessary for activation of fast muscle genes by MyoD (\cite{Maves2007}) and cardiac expression of Hand2 (\cite{Maves2009}). This results support the proposed 'pioneer' function for Pbx (\cite{Berkes2004}) by which it would pre-occupy closed chromatin and enable its targeting by chromatin remodelling-inducing factors, setting the range of permissible sites for these factors. Such a pioneer functions has also been proven for Pbx regarding the estrogen receptor (\cite{Magnani2011}). %genetic interaction prep-meis-pbx

\subsection{Involvement in Leukemias}

While Meis1 alone cannot induce leukemia, joint overexpression of Meis1 and Hoxa7 or Hoxa9 is sufficient to induce myeloid leukemia (\cite{Nakamura1996, Schnabel2000}).

The E2a-Pbx1 fusion protein resulting from the t(1;19) translocation induces pre-B cell acute lymphoblastic leukemia (\cite{Kamps1990}). This fusion protein is able to bind DNA and to interact with Hoxes, but not with Meis or Prep. The transcriptional activation domain from E2a turns it into a constitutional activator. 

The portion of \ac{prepi} embryos that survive development and reach adulthood develop spontaneous tumours or pretumoral lesions, revealing a role for Prep1 as a tumour-suppressor gene (\cite{Longobardi2010}).

\subsection{Other Functions}

The \ac{TALE} proteins have also been implicated in limb \ac{PD} patterning. Meis1 and Meis2 are expressed in the proximal portion of the vertebrate limb and their ectopic expression in distal limb induces nuclear translocation of normally cytoplasmic Pbx and distal to proximal transformations (\cite{Mercader1999b}). Meis is downstream of \ac{RA}, which is produced by the somites and present in the limb in a \ac{PD} gradient, opposing the apical ectodermal ridge-produced FGF gradient (\cite{Mercader2000, Rosello-Diez2011}).

Meis and Pbx are involved in cardiac development, as shown by semilunar valve, ventricular septum and outflow tract defects in Meis and Pbx mutants (\cite{Stankunas2008}). Recently, Meis has been shown to be expressed and recruited to enhancers during \textit{in vitro} differentiation of cardiomyocytes, specifically at the cardiac progenitor stage and apparently in combination with GATA factors (\cite{Wamstad2012}). Meis2 knockdown in Zebrafish results in failure of cardiac tube looping (\cite{Paige2012}). 

Meis has also been recently implicated in mammalian neonatal heart regeneration through cardiomyocyte mitotic arrest (\cite{Mahmoud2013}). Specific deletion of Meis1 in cardiomyocytes resulted in increased postnatal cardiomyocyte proliferation while overexpression resulted in decreased proliferation. Reporter constructs showed regulation of Ink4b–Arf–Ink4a and p21 by Meis.

% reporter transcriptional activation in synergy with Gata4 was specific for Meis1a (short \ac{C-ter})rather than Meis2d (long \ac{C-ter}) (\cite{Wamstad2012}) 

% [genomic stability]
\section{ChIP-seq}


The \ac{CHIP} assay consists of cross-linking of chromatin to preserve the non-covalent interactions between proteins and DNA for biochemical assay. It was first described by \textcite{Solomon1988}. For many years its application remained confined to focused application \parencite{Mardis2007}.

The first genome-wide application of the technique was achieved by microarray hibridization for determination of DNA fragment enrichment (ChIP-chip, \cite{Ren2000}). 

As massive sequencing has gone down in cost, its application to assaying ChIP-enriched DNA fragments has increased. The ChIP-seq technique (\cite{Robertson2007}) involves deep sequencing of DNA fragments followed by bioinformatic treatment of the data to extract genomic regions bound by the factor of interest (peaks). It provides better precision and less bias than ChIP-chip (see for example \cite{Rhee2011}) and has allowed the \ac{GW} determination of \ac{TF} \acp{BS} and histone  marks in an unprecedented number of cell types and tissues, for example in the ENCODE project (\cite{2004, 2007, 2012}).



%[meter Deep homology (eg tinman-nkx2.5, hth-Meis, Pax6-eyeless)?]
