\chapter{Results \& Discussion}
\label{chp:results}

\section{Cataloging, describing Meis, Prep, Pbx1 ChIP-seq peaks}

\subsection{\# reads, sequencing, mapping info}
Chromatin Immunoprecipitation was performed in the dissected trunks of embryonic stage E11.5 C57BL/6 mice.





\subsection{Peak Calling}

We initially attempted to call peaks from read data with the widely used \ac{MACS} algorithm (\cite{MACS ref}). However, precision was very low, i.e. peaks were about 2000pb wide. Visual inspection of the read profile revealed that, even when the enrichment was clearly delimited an an obvious peak was visible, the peak regions called by \ac{MACS} extended far beyond the summit and even the base of the peak. The mean peak profile for the highest enriched peaks is generated by \ac{MACS} in the process of defining an "archetypal" peak, which is later used as a reference to score read accumulations. This profile, in our case, included obviously extraneous adjacent regions where the read density was at genomic background levels [meter \ref{fig:MACSpeakProfile}]. No combination of parameters could refine the peak calling. We speculate that this glitch might be due to the read numbers used in contemporary deep sequencing experiments, much higher than those common when \ac{MACS} was designed.

We then used the \ac{PICS} algorithm \cite{ref} for calling peaks from read data. \ac{PICS} produced much more precise peak calls than \ac{MACS}. These were used for all following analyses.

\subsection{Overlap of Meis, Prep and Pbx1 peaks}

5686 peaks were found for Meis, 3331 for Prep, and 3504 for Pbx1, for a total of 10326 non-redundant genomic regions. The overlaps between the three sets are represented in Figure \ref{fig:peakOverlap}. The first conclusion from this analysis is that Meis overlaps much less than expected with either Prep or Pbx1. 4837 of 5686 (85\%) Meis peaks are \ac{meise}. Of the remaining 15\%, 186 (3.3\%) are \ac{mepr}, 444 (7.8\%) were \ac{mepb}, and 222 (3.9\%) are bound by all three factors. 

Prep and Pbx1 peaks overlap much more with each other than with Meis. Of 3331 Prep peaks, 57\% are \ac{prepe},  5.6\% are \ac{mepr}, 30.7\% are \ac{pbpr} and the 222 triple peaks represent the remaining 6.7\%. For the 3504 Pbx1 peaks, 1821 (52\%) are \ac{pbxe}, 444 (12.7\%) are \ac{mepb}, 1023 (29.2\%) are \ac{pbpr} and triple peaks represent 6.3\%. 


%[por que? a lo mejor el anticuerpo no esta reconociendo Pbx con Meis? que epitopo reconoce? ha sido usado para ver triples complejos?]


\begin{figure}[]
  
  \centering
  \label{fig:peakOverlap}
  \includegraphics[width=\textwidth]{figures/vennDiagram_peakOverlap}
  \caption[Overlap of Meis, Prep, and Pbx1 peaks]{\textbf{Overlap of Meis, Prep, and Pbx1 peaks.} Venn diagram of the number of \ac{meise},\ac{prepe}, \ac{pbxe}, \ac{mepr}, \ac{mepb} and \ac{mepr} peaks. Areas of the circles are proportional to total peak numbers for Meis (red), Prep (blue) and Pbx1 (green) peaks. Intersection areas are roughly proportional to the number of \ac{mepr}, \ac{mepb}, \ac{mepr} and triple peaks.}
\end{figure}



\subsubsection{Triple peak Chip-reChip}

Triple peaks show the most obvious immediate interest, since they might represent \acp{CRM} integrating inputs from all three factors. The experimental result that three different factors are able to bind DNA on the same genomic site can have two interpretations. The factors might bind simultaneously in the form of a complex or to very close \acp{BS} on the same \ac{CRM}. Alternatively, since our input chromatin is a nuclear extract from many different cell types, the factors might bind alternatively, maybe competing for the same target sequences.

To distinguish between the two possibilities, we performed ChIP-re-ChIP experiments. In these, the output of a \ac{CHIP} is used as input for a further round of \ac{CHIP} with antibodies against a different protein. In this way, the resulting DNA will be enriched specifically in fragments bound by the two proteins simultaneously. Results are shown in Figure \ref{fig:chiprechip}.

\begin{figure}[]
  
  \centering
  \label{fig:chiprechip}
  \includegraphics[width=.5\textwidth]{figures/chiprechip}
  \caption[ChIP-re-ChIP experiments]{\textbf{ChIP-re-ChIP experiments.} Factors were subjected to succesive round of \ac{CHIP} with antibodies with different specificities, as indicated. Unspecific antibody (IgG) was used as a negative control. Purified DNA was amplified in a rate-limiting \ac{PCR} to obtain an amount of product roughly proportional to the peak DNA present. Band intensities over IgG band were considered as positive signals.}
\end{figure}

We tested 6 double peaks and 16 triple peaks. All of them showed simultaneous binding by at least two factors. The most frequent co-binding combinations we saw were either Pbx1+Prep (10/17) or Pbx1+Meis (17/21). In the 16 triple peaks we tested, we saw Meis+Prep co-binding in only 2 (Meis peaks 56 and 3034). One of these [completar]

These results suggest that Prep and Meis very rarely bind to the same genomic site at the same time. Therefore, the 222 triple peaks probably represent instances of competition between Meis and Prep for binding to Pbx1 and DNA. [ampliar]


\subsection{Distance to TSSs}

The first step in the genomic characterization of peaks was to find the \ac{TSS} nearest to each peak. The gene set we used for this and all following gene-centric analyses was Ensembl63, the latest version available at the time of mapping.

Distance between peak summit and \ac{TSS} was calculated using the \ac{TSS} as position 0 and direction of transcription as + direction, in order to differentiate upstream and downstream peaks. Results are represented in Figure \ref{fig:distanceHistogram}. We found a clear concentration of peaks near \ac{TSS}s for both Prep and Pbx1. Meis peaks, in contrast, show very little concentration near \ac{TSS}s, and a much higher proportion of them is located at extreme distances from \ac{TSS}s. 

\begin{figure}[]
  
  \centering
  \label{fig:distanceHistogram}
  \includegraphics[width=\textwidth]{figures/distanceHistogram}
  \caption[Distance From Peaks to \ac{TSS}s]{\textbf{Distance From Peaks to \ac{TSS}s.} The percentage of peaks in each category over total factor peaks is represented in the Y axis. X axis categories are labelled with the right limit of the interval, in \ac{bp}.}
\end{figure}




\subsection{TSSA, IG, CI, FI}

In order to understand better the interplay between peak overlap and distance to transcriptional units, we classified peaks into \ac{TSSA}, \ac{IG}, \ac{CI} or \ac{FI}. The criteria for classification were based on the  distances represented in Figure \ref{fig:distanceHistogram}, in which the concentration near \acp{TSS} is very clearly delimited between -500 and +100, and the divergence in proportion between promoter-remote Meis and Prep or Pbx peaks starts at around 20kb. Another reason for the subdivision of intergenic peaks is the difficulty of assigning potential transcriptional targets to peaks that are at extreme distances from the nearest \ac{TSS}.

Therefore, peaks located between -500 and +100 were considered to be \ac{TSSA}. Of the rest, those within the gene body were considered \ac{IG}. The peaks located in intergenic regions were divided into \ac{CI} (those outside the transcriptional unit but within 20kb of the nearest \ac{TSS}) and \ac{FI} (further than 20kb away from the nearest \ac{TSS}). 
[comentar el pico en Prep entre 100 y 200]

\begin{figure}[]
  
  \centering
  \label{fig:distanceBarChart}
  \includegraphics[width=\textwidth]{figures/barChart_TSSAIGCIFI}
  \caption[Genomic location classes within each factor-binding profile category]{\textbf{Genomic location classes within each factor-binding profile category.} The peaks for each combination of binding factors were classified into \ac{TSSA}, \ac{IG}, \ac{CI} or \ac{FI} categories. The resulting profile is shown here. ALL = total non-redundant genomic sites. Pr = \ac{prepe}, M = \ac{meise}, P = \ac{pbxe}, Pr-M = \ac{mepr}, Pr-P = \ac{pbpr}, M-P = \ac{mepb}, Pr-M-P = triple peaks.}
\end{figure}

The results of this classification are represented in Figure \ref{fig:distanceBarChart}. There are obvious differences specially in the proportion of \ac{TSSA} peaks bound by different combinations of factors. Meis and Pbx1 bind \acp{TSS} very rarely, either alone or in combination. In contrast, over 30\% of \ac{prepe} peaks are located near a \ac{TSS}, a proportion that rises to over 65\% when bound in combination with Pbx1 (\ac{pbpr} peaks).

Intergenic peaks formed the majority of \ac{meise} peaks: over 45\% of \ac{meise} are \ac{FI} [completar]

[meter porcentajes]

\subsubsection{Intron/Exon Ratio of Meis Peaks is skewed}

We considered the \ac{IG} peaks. 39.3\% of the mouse genome is covered by introns while 2\% is covered by exons, which results in an intron/exon ratio of 19.65. The corresponding percentages for Meis (39.9\% versus 0.5\%), Prep (24.9\% versus 1.6\%) and Pbx1 (31.4\% versus 1.6\%) work out to ratios of 79.8, 15.56 and 19.63, respectively. The difference was highly significant for Meis (p-value < $10^20$) but not for Prep or Pbx1. 


\subsection{Conservation Profiling}

We analyzed the conservation profile of peaks. For this analysis, the peak summit was used as position 0 and the phastCons (\cite{Siepel2005}) score was averaged for equivalent positions of all peaks of each subset. PhastCons is a measure of phylogenetic conservation that varies between 0 and 1 and roughly represents the probability that a given base is a conserved element. Phastcons is estimated from a hidden Markov model based on multiple sequence alignments and is available precomputed in several genomic databases. The version we used was 30-vertebrate phastCons.

The resulting profile shows that Prep and Pbx1 peaks have a high conservation value near the summit that tails off to genomic average values. The maximum conservation value and the profile are very similar to \acp{TSS} marked by \ac{PolII} peaks from \cite{Mahony2011}, which together with the previously mentioned high proportion of \ac{TSSA} peks within Prep and Pbx1 suggests a significant portion of these may be enhancers.

Meis peaks have a much higher conservation profile, approaching genomic regions bound by p300 in the embryonic forebrain. These p300 peaks represent the most conserved enhancers (\cite{Blow2010}). The only other \ac{TF} peaks approaching this level of conservation that we have observed are, notably, those for HoxC9 in the embryonic spinal cord \cite{Jung2010}. Thus, genomic loci bound by Meis are both far way from \acp{TSS} and highly conserved, which strongly suggests functional relevance. In particular, they might be working as enhancers.

[what does HoxA2 look like?] 

\begin{figure}[]
  
  \centering
  \label{fig:conservation}
  \includegraphics[width=\textwidth]{figures/conservation}
  \caption[Conservation Profile of Peaks]{\textbf{Conservation Profile of Peaks.} PhastCons values at each position from summit were averaged across all peaks in a subset and plotted. Peaks for Meis, Prep, and Pbx1 are from this study. Peaks for HoxC9 are from \cite{Jung2010}. Peaks for p300 in mouse embryonic forebrain were extracted from \cite{Blow2010}. Peaks for PolII are from \cite{Mahony2011}. Other TFs in grey are CEBPA and HNF4A (\cite{Schmidt2010}), RAR (\cite{Mahony2011}) and FLAG-tagged Meis1 and HoxA9 in a leukemia cell line (\cite{Huang2012}).}
\end{figure}


[meto los histogramas de valores de conservaciÃ³n para todo el pico?]

\subsection{Histone Marks}

In order to perform a test of this potential function of Meis \acp{BS} as enhancers, we checked their association with histone post-translational modifications. Different histone marks are associated with active and inactive promoters and enhancers, and can therefore be used to indicate chromatin state (\cite{Mikkelsen2007}). In particular, RNApolII\textsuperscript{+} H3K4Me3\textsuperscript{+} marks promoters,  H3K4Me1\textsuperscript{+} H3K4Me3\textsuperscript{-} marks enhancers and H3K4Me1\textsuperscript{+} H3K27Ac\textsuperscript{+} marks bivalent enhancers. 

We took advantage of the extensive, high-quality epigenetic data generated by the Ren lab at LICR for the ENCODE project (\cite{Shen2012}). We chose the source tissue that most closely resembled our sample, \acp{MEF}. 

For all three factors, \ac{TSSA} peaks showed a high level of overlap with promoter epigenetic marks (between 43.8 and 58.6\% of all \ac{TSSA} peaks versus 0.38\% of the genome tagged RNApolII\textsuperscript{+} H3K4Me3\textsuperscript{+}). Only Prep non-\ac{TSSA} peaks showed high association with promoter marks (around 17\% for all other categories). Meis non-\ac{TSSA} peaks had low levels of coincidence (between 1.7 and 4.2\%) and Pbx1 non-\ac{TSSA} peaks had levels intermediate between those of Meis and Prep (between 5.6 and 9.8\%) (Figure \ref{fig:histoneMarks}a). Thus we conclude that \ac{TSSA} peaks likely represent \textit{bona fide} promoters. In addition, many non-\ac{TSSA} Prep peaks could in fact be either undescribed promoters or simply promoters not present in the gene catalogue we used.

In contrast with promoter marks, enhancer (H3K4Me1\textsuperscript{+} H3K4Me3\textsuperscript{-}) and bivalent enhancer (H3K4Me1\textsuperscript{+} H3K27Ac\textsuperscript{+}) marks are much more associated with Meis peaks than with Prep or Pbx1 peaks. 25.5\% of Meis peaks bear enhancer marks versus [completar]





\begin{figure}[]
  
  \centering
  \label{fig:histoneMarks}
  \includegraphics[width=\textwidth]{figures/barChart_histoneMarks}
  \caption[Histone Marks in Peak Subsets]{\textbf{Histone Marks in Peak Subsets.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}


 
 
\section{Sequence analysis}

\subsection{Motif discovery stats, core motifs}




\subsection{distribution in different subsets}

\begin{figure}[]
    \centering
  \label{fig:coreMotifsFound}
  \includegraphics[width=\textwidth]{{{figures/coreMotifs.found}}}
  \caption["Core" Motifs Found in Peak subsets]{\textbf{"Core" Motifs Found in Peak subsets.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}

\end{figure}


\subsection{Comparison with expected distribution of motifs}

\begin{figure}[]
  
  \centering
  \label{fig:coreMotifsExpected}
  \includegraphics[width=\textwidth]{{{figures/coreMotifs.expected}}}
  \caption["Core" Motifs Expected in Peak subsets]{\textbf{"Core" Motifs Expected in Peak subsets.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
\end{figure}

\subsection{Hox motif variants, distribution}

[can we guess the preferred hoxs for Meis based on the distribution of target sequences?]

\subsection{uncore motifs}

\section{Transcriptional targets}

\subsection{\# reads, sequencing, mapping info}

\subsection{attempting to correlate}

\subsection{logistical model (?)}

\section{Hox Clusters}

\subsection{Distribution in the Hox clusters}

\subsection{AP ChIP}

\subsection{previously described reg sites}

\subsection{Hox RNAseq}


[por colocar:

- comparacion detallada con Meis ChIP-seq hematop {Wilson et al. 20101, Huang et al. 2012}
- Chip-proteomics
]
