 \chapter{Results}
\label{chp:results}

\section{Characterization of TALE protein ChIP-seq Peaks}

\subsection{\# reads, sequencing, mapping info}
\ac{CHIP} was performed in the dissected trunks of embryonic stage E11.5 C57BL/6 mice. The trunk was selected as input material because it is the main domain of Meis and Hox expression.


%[MT: Describe first the antibodies and what are they expected to pull down.  Describe the comprehesiveness of the experiment for Meis. Prep and Pbx proteins.  describe how many reads were obtained and aligned]

\subsection{Peak Calling}

We initially attempted to call peaks from read data with the widely used \ac{MACS} algorithm (\cite{MACS ref}). However, precision was very low, i.e. peaks were about 2000pb wide. Visual inspection of the read profile revealed that, even when the enrichment was clearly delimited and an obvious peak was visible, the peak regions called by \ac{MACS} extended far beyond the summit and even the base of the peak. The mean peak profile for the highest enriched peaks is generated by \ac{MACS} in the process of defining an "archetypal" peak, which is later used as a reference to score read accumulations. This profile, in our case, included obviously extraneous adjacent regions where the read density was at genomic background levels [meter figura \ref{fig:MACSpeakProfile}]. No combination of parameters could refine the peak calling. We speculate that this glitch might be due to the read numbers used in contemporary deep sequencing experiments, much higher than those common when \ac{MACS} was designed.

We then used the \ac{PICS} algorithm \cite{ref} for calling peaks from read data. \ac{PICS} produced much more precise peak calls than \ac{MACS}. These were used for all following analyses.

\subsection{Overlap of Meis, Prep and Pbx1 peaks}

5686 peaks were found for Meis, 3331 for Prep, and 3504 for Pbx1, for a total of 10326 non-redundant genomic regions. The overlaps between the three sets are represented in Figure \ref{fig:peakOverlap}. The first conclusion from this analysis is that Meis overlaps much less than expected with either Prep or Pbx1. 4837 of 5686 (85\%) Meis peaks are \ac{meise}. Of the remaining 15\%, 186 (3.3\%) are \ac{mepr}, 444 (7.8\%) were \ac{mepb}, and 222 (3.9\%) are bound by all three factors. 
%[MT: Mention that pbx screen is not comprehensive while prep and meis mostly is.]

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/vennDiagram_peakOverlap}
  \caption[Overlap of Meis, Prep, and Pbx1 peaks]{\textbf{Overlap of Meis, Prep, and Pbx1 peaks.} Venn diagram of the number of \ac{meise},\ac{prepe}, \ac{pbxe}, \ac{mepr}, \ac{mepb} and \ac{mepr} peaks. Areas of the circles are proportional to total peak numbers for Meis (red), Prep (blue) and Pbx1 (green) peaks. Intersection areas are roughly proportional to the number of \ac{mepr}, \ac{mepb}, \ac{mepr} and triple peaks.}
  \label{fig:peakOverlap}
\end{figure}

Prep and Pbx1 peaks overlap much more with each other than with Meis. Of 3331 Prep peaks, 57\% are \ac{prepe},  5.6\% are \ac{mepr}, 30.7\% are \ac{pbpr} and the 222 triple peaks represent the remaining 6.7\%. For the 3504 Pbx1 peaks, 1821 (52\%) are \ac{pbxe}, 444 (12.7\%) are \ac{mepb}, 1023 (29.2\%) are \ac{pbpr} and triple peaks represent 6.3\%. 

%[por que? a lo mejor el anticuerpo no esta reconociendo Pbx con Meis? que epitopo reconoce? ha sido usado para ver triples complejos?]

\subsubsection{Triple peak Chip-reChip}

Triple peaks show the most obvious immediate interest, since they might represent \acp{CRM} integrating inputs from all three factors. The experimental result that three different factors are able to bind DNA on the same genomic site can have two interpretations. The factors might bind simultaneously in the form of a complex or to very close \acp{BS} on the same \ac{CRM}. The other possibility is that, since our input chromatin is a nuclear extract from many different cell types, the factors might bind alternatively, maybe competing for the same target sequences.

\begin{SCfigure}[]
  \centering
  \includegraphics[width=.6\textwidth]{figures/chiprechip}
  \caption[ChIP-re-ChIP experiments]{\textbf{ChIP-re-ChIP experiments.} Factors were subjected to succesive round of \ac{CHIP} with antibodies with different specificities, as indicated. Unspecific antibody (IgG) was used as a negative control. Purified DNA was amplified in a rate-limiting \ac{PCR} to obtain an amount of product roughly proportional to the peak DNA present. Band intensities over IgG band were considered as positive signals.}
  \label{fig:chiprechip}
\end{SCfigure}

To distinguish between the two possibilities, we performed ChIP-re-ChIP experiments. In these, the output of a \ac{CHIP} is used as input for a further round of \ac{CHIP} with antibodies against a different protein. In this way, the resulting DNA will be enriched specifically in fragments bound by the two proteins simultaneously. Results are shown in Figure \ref{fig:chiprechip}.

We tested 6 double peaks and 16 triple peaks. All of them showed simultaneous binding by at least two factors. The most frequent co-binding combinations we saw were either Pbx1+Prep (10/17) or Pbx1+Meis (17/21). In the 16 triple peaks we tested, we saw Meis+Prep co-binding in only 2 (Meis peaks 56 and 3034). One of these %[completar]

These results suggest that Prep and Meis very rarely bind to the same genomic site at the same time. Therefore, the 222 triple peaks probably represent instances of competition between Meis and Prep for binding to Pbx1 and DNA. %[ampliar]


\subsection{Position of Bindings Sites Relative to Transcription Start Sites}

The first step in the genomic characterization of peaks was to find the \ac{TSS} nearest to each peak. The gene set we used for this and all following gene-centric analyses was Ensembl63, the latest version available at the time of mapping.

\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/distanceHistogram}
  \caption[Distance From Peaks to \ac{TSS}s]{\textbf{Distance From Peaks to \ac{TSS}s.} The percentage of peaks in each category over total factor peaks is represented in the Y axis. X axis categories are labelled with the right limit of the interval, in \ac{bp}.}
  \label{fig:distanceHistogram}
\end{figure}

Distance between peak summits and \acp{TSS} was calculated using the \ac{TSS} as position 0 and direction of transcription as + direction, in order to differentiate upstream and downstream peaks. Results are represented in Figure \ref{fig:distanceHistogram}. We found a clear concentration of peaks near \ac{TSS}s for both Prep and Pbx1. Meis peaks, in contrast, show very little concentration near \acp{TSS}, and a much higher proportion of them is located at extreme distances from \acp{TSS}. % [MT: Is the enrichment in distant 3' regions vs 5' regions significant?  if yes, how can we explain this?]

\subsection{TSSA, IG, CI, FI}

In order to understand better the interplay between peak overlap and distance to transcriptional units, we classified peaks into \ac{TSSA}, \ac{IG}, \ac{CI} or \ac{FI}. The criteria for classification were based on the  distances represented in Figure \ref{fig:distanceHistogram}, in which the enrichment in Prep peaks near \acp{TSS} is very clearly delimited between -500 and +100. 

We subdivided intergenic peaks into \ac{CI} and \ac{FI} because of the the difficulty of assigning potential transcriptional targets to peaks that are at extreme distances from the nearest \ac{TSS}.% and the divergence in proportion between promoter-remote Meis and Prep or Pbx peaks starts at around 20kb. 

Therefore, peaks located between -500 and +100 were considered to be \ac{TSSA}. Of the rest, those within the gene body were considered \ac{IG}. The peaks located in intergenic regions were divided into \ac{CI} (those outside the transcriptional unit but within 20kb of the nearest \ac{TSS}) and \ac{FI} (further than 20kb away from the nearest \ac{TSS}). 
%[comentar el pico en Prep entre 100 y 200]

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/barChart_TSSAIGCIFI}
  \caption[Genomic location classes within each factor-binding profile category]{\textbf{Genomic location classes within each factor-binding profile category.} The peaks for each combination of binding factors were classified into \ac{TSSA}, \ac{IG}, \ac{CI} or \ac{FI} categories. The resulting profile is shown here. ALL = total non-redundant genomic sites. Pr = \ac{prepe}, M = \ac{meise}, P = \ac{pbxe}, Pr-M = \ac{mepr}, Pr-P = \ac{pbpr}, M-P = \ac{mepb}, Pr-M-P = triple peaks.}
  \label{fig:distanceBarChart}
\end{figure}

The results of this classification are represented in Figure \ref{fig:distanceBarChart}. There are obvious differences specially in the proportion of \ac{TSSA} peaks bound by different combinations of factors. Meis and Pbx1 bind \acp{TSS} very rarely, either alone or in combination. In contrast, over 30\% of \ac{prepe} peaks are located near a \ac{TSS}, a proportion that rises to 71.5\% when bound in combination with Pbx1 (\ac{pbpr} peaks). 

\ac{mepr} peaks show a distribution intermediate between those of \ac{meise} and \ac{prepe} peaks.

Intergenic peaks form the majority of \ac{meise} peaks: 45.7\% of \ac{meise} peaks are \ac{FI}. \ac{pbxe} and \ac{mepb} peaks show a profile very similar to that of \ac{meise} peaks. Triple peaks have a flat distribution in theese four categories.
% [MT: What is the abundance of these classes in the genome?  maybe we could include this for comparison of what would be a neutral distribution of peaks]
\subsubsection{Intron/Exon Ratio of Meis Peaks is skewed}

We considered the \ac{IG} peaks. 39.3\% of the mouse genome is covered by introns while 2\% is covered by exons, which results in an intron/exon ratio of 19.65. The corresponding percentages for Meis (39.9\% versus 0.5\%), Prep (24.9\% versus 1.6\%) and Pbx1 (31.4\% versus 1.6\%) work out to ratios of 79.8, 15.56 and 19.63, respectively. The difference was highly significant for Meis (p-value \< $10^20$) but not for Prep or Pbx1 (both p-values over $10^{-1}$). 

\subsection{Conservation Profiling}

We analyzed the evolutionary conservation profile of peaks. For this analysis, the peak summit was used as position 0 and the phastCons (\cite{Siepel2005}) score was averaged for equivalent positions of all peaks of each subset. PhastCons is a measure of phylogenetic conservation that varies between 0 and 1 and roughly represents the probability that a given base is a conserved element. Phastcons is estimated from a hidden Markov model based on multiple sequence alignments and is available precomputed in several genomic databases. The version we used was 30-vertebrate phastCons.

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/conservation}
  \caption[Conservation Profile of Peaks]{\textbf{Conservation Profile of Peaks.} PhastCons values at each position from summit were averaged across all peaks in a subset and plotted. Peaks for Meis, Prep, and Pbx1 are from this study. Peaks for HoxC9 are from \cite{Jung2010}. Peaks for p300 in mouse embryonic forebrain were extracted from \cite{Blow2010}. Peaks for PolII are from \cite{Mahony2011}. Other TFs in grey are CEBPA and HNF4A (\cite{Schmidt2010}), RAR (\cite{Mahony2011}) and FLAG-tagged Meis1 and HoxA9 in a leukemia cell line (\cite{Huang2012}).}
  \label{fig:conservation}
\end{figure}

The resulting profile shows that Prep and Pbx1 peaks have a high conservation value near the summit that tails off to genomic average values. The maximum conservation value and the profile are very similar to \acp{TSS} marked by \ac{PolII} peaks from \cite{Mahony2011}, which together with the previously mentioned high proportion of \ac{TSSA} peks within Prep and Pbx1 suggests a significant portion of these may be promoters.

Meis peaks have a much higher conservation profile, approaching that of genomic regions bound by p300 in the embryonic forebrain. These p300 peaks represent the most conserved enhancers (\cite{Blow2010}). The only other \ac{TF} peaks approaching this level of conservation that we have observed are, notably, those for HoxC9 in the embryonic spinal cord \cite{Jung2010}. Thus, genomic loci bound by Meis are both far way from \acp{TSS} and highly conserved, which strongly suggests functional relevance.

[what does HoxA2 look like?] 

[meto los histogramas de valores de conservación para todo el pico?]

\subsection{Relationship of TALE Factor Binding Sites with Histone Marks}

In order to study the correlation between the peak sets identified and known functional genomic regulatory elements, we checked their association with histone post-translational modifications. Different histone marks are associated with active and inactive promoters and enhancers, and can therefore be used to indicate chromatin state (\cite{Mikkelsen2007}). In particular, RNApolII\textsuperscript{+} H3K4Me3\textsuperscript{+} marks promoters,  H3K4Me1\textsuperscript{+} H3K4Me3\textsuperscript{-} marks enhancers and H3K4Me1\textsuperscript{+} H3K27Ac\textsuperscript{+} marks bivalent (poised) enhancers (\cite{Creyghton2010}). 

We took advantage of the extensive, high-quality epigenetic data generated by the Ren lab at LICR for the ENCODE project (\cite{Shen2012}). We chose the source tissue that most closely resembled our sample, \acp{MEF}. 

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/barChart_histoneMarks}
  \caption[Histone Marks in Peak Subsets]{\textbf{Histone Marks in Peak Subsets.} A) percentage of peaks in each genomic binding class that show promoter (RNApolII\textsuperscript{+}; H3K4Me3\textsuperscript{+}) epigenetic marks. B) percentage of peaks for each factor that show enhancer (H3K4Me1\textsuperscript{+}; H3K4Me3\textsuperscript{-}) or bivalent enhancer (H3K4Me1\textsuperscript{+}; H3K27Ac\textsuperscript{+}) histone marks.}
  \label{fig:histoneMarks}
\end{figure}

For all three factor peak sets, \ac{TSSA} peaks showed a high level of overlap with promoter epigenetic marks (between 43.8 and 58.6\% of all \ac{TSSA} peaks versus 0.38\% of the genome tagged RNApolII\textsuperscript{+} H3K4Me3\textsuperscript{+}). Only Prep non-\ac{TSSA} peaks showed high association with promoter marks (around 17\% for all other categories). Meis non-\ac{TSSA} peaks had low levels of coincidence (between 1.7 and 4.2\%) and Pbx1 non-\ac{TSSA} peaks had levels intermediate between those of Meis and Prep (between 5.6 and 9.8\%) (Figure \ref{fig:histoneMarks}a). Thus we conclude that \ac{TSSA} peaks likely represent \textit{bona fide} promoters. In addition, many non-\ac{TSSA} Prep peaks could in fact be either undescribed or cryptic promoters or simply promoters not present in the gene catalogue we used.

In contrast with promoter marks, enhancer (H3K4Me1\textsuperscript{+} H3K4Me3\textsuperscript{-}) and bivalent enhancer (H3K4Me1\textsuperscript{+} H3K27Ac\textsuperscript{+}) marks are much more associated with Meis peaks than with Prep or Pbx1 peaks. 25.5\% of Meis peaks bear enhancer marks versus [completar]
 
\section{Sequence analysis}

Beyond the genomic location of the \acp{BS} of our factors of interest, we sought to characterize the sequences they select \textit{in vivo}, and then contrast the preferences of each factor and compare them to their \textit{in vitro} behaviour. 

\subsection{Motif discovery}

We performed motif discovery on the peak sequences. In short, motif discovery is a way to discover short sequences that are more frequent in a sequence dataset than would be expected by chance. We used the rGADEM software, an implementation of the GADEM (\cite{Li2009a}) algorithm in the R statistical computing language that integrates smoothly with \ac{PICS} and produces a detailed and readable output.

We ran rGADEM [parameters] on the sequence of non-overlapping subsets of peaks defined by their factor combination: \ac{meise},\ac{prepe}, \ac{pbxe}, \ac{mepr}, \ac{mepb}, \ac{mepr} and triple peaks, separately. The results are shown in Figure \ref{fig:allMotifs}.

\subsection{Core Versus Accessory Motifs}

\begin{SCfigure}[]
    \centering
  \includegraphics[width=.7\textwidth]{figures/allMotifs}
  \caption[All Motifs Found in Peak subsets]{\textbf{All Motifs Found in Peak subsets.} Sequence logos for every motif found in each non-overlapping peak subset. "Core" motifs labelled in red. The logos resulting from the motif definitions (\acp{PSSM}) which we selected as canonical and used for directed motif search are highlighted in grey. The histograms to the right of the sequence logo show the distribution of motif positions with respect to the summit of the peak.}
  \label{fig:allMotifs}
\end{SCfigure}

The representation of motif distances to the peak summit shows a fairly clear distinction between two different motif classes: "core" motifs that tend to be located near the summit of the peak and therefore shown a unimodal distance distribution, and "accessory" motifs that are depleted near the summit of the peak and thus show a bimodal distribution. 

Core motifs are likely to be \textit{bona fide} factor recognition sequences. The fact that they tend to be located at or close to the maximum confidence point for factor binding, the summit, suggests that they are directly responsible for this binding.

In contrast, accessory motifs may be interpreted in various ways. Accessory motifs might represent \acp{BS} for cofactors that collaborate in the binding. Alternatively, they could represent local DNA compositional biases that facilitate the binding of the factors without interacting with them, for example by imparting a certain curvature or groove topology to the stretch of DNA. They might also be simply artifacts: if a subset of peaks is located preferentially in genomic regions with particular DNA sequence characteristics, one would expect those characteristics to be reflected in the results of motif discovery, although they might not have a direct effect on factor binding. % [MT: Mencionar que algunos son bastante sharp, indicando posiblemente una distancia preferida y que por lo tanto son genuinos y problablemente unan algun factor.  indicar qué factores podrian unirse a estas secuencias accesorias]

[motif discovery on promoter peaks regardless of factor: is there any particular motif that shows up?]


\subsection{Core Motifs}

Most of the core motifs we found are immediately recognizable from the literature on Hox cofactor DNA binding preferences and were identified in several peak subsets. The motif we call \ac{HEXA} corresponds to the canonical monomeric Meis/Prep \ac{BS} and is detected in the \ac{meise} and \ac{mepb} subsets, but surprisingly not in the \ac{prepe} subset. The \ac{OCTA} fits the well-known Pbx-Hox heterodimeric \ac{BS} and is present in all subsets with Meis binding, even in those peaks in which we have not found evidence of Pbx binding, such as \ac{meise} and \ac{mepr}. The \ac{DECA} clearly resembles a heterodimeric Pbx-Meis or Pbx-Prep \ac{BS} with a 5' TGAT Pbx half and a 3' Prep/Meis TGACAG half. We found an additional previously unreported bipartite motif that contains a \ac{DECA} and a CCAAT sequence at a fixed distance, which we have called \ac{DECAEXT}. \ac{DECAEXT} only appears in motif discovery on \ac{pbpr} peaks. In \ac{pbxe} peaks we only found one low complexity core motif that did not match any found in the literature for Hox or \ac{TALE} proteins, but might be related to accessory motifs found in other subsets (see below). Because of its poor definition, we discarded it from further analysis. % [MT: Comment on motifs detected in triple factor peaks]

\subsection{Accessory Motifs}
\label{ssc:accessoryMotifs}

Several intriguing sequences appear as accessory motifs. All subsets show variants of a poorly defined sequence composed of Gs and As in no particular order: motif 4 in \ac{meise}, 3 and 4 in \ac{prepe}, 5 in \ac{pbxe}, 3 in \ac{mepb}, 5 in \ac{mepb}, 4 in \ac{pbpr}, 3 and 4 in triple peaks. It is interesting to note that this represents DNA stretches in which one strand is exclusively populated by purines and the complementary strand is only populated by pyrimidines. Poly-purine tracts have been shown to be able to form DNA triple helical structures \textit{in vitro} (\cite{ref}), to be implicated in splicing (\cite{ref}), and are a known component of retroviruses (\cite{ref})[completar, explorar, fill refs].

Variants of a CTGnCTG sequence appear in many of the subsets. In \ac{meise} peaks, motifs 3 and 5 to 8 all fit this consensus, as do motifs 1, 3 and 6 from \ac{pbxe} peaks. In the \ac{mepb} subset, motif 6 and the reverse complement of motif 4 could represent variants of this consensus. This motif was found in a previous report (\cite{Tijssen2011}) and described as an "motif resembling an E-box". E-boxes are binding sites for \ac{BHLH} proteins, and the myogenic \ac{BHLH} proteins have been reported to be able to bind DNA with Pbx-Meis heterodimers (\cite{Knoepfler1999, Berkes2004}).

Another of the accessory motifs appeared in several subsets and has suggestive connections in the literature. a (C)$_n$N(C)$_n$ motif where n is between 3 and 4 appears in the \ac{prepe}, \ac{mepb} and \ac{pbpr} subsets. It is very similar to the known Sp1 binding site (GGGGnGGGG, \cite{ref}). Sp1 is [leer, completar]

\subsection{Comparison with Expected Distribution of Motifs}

Going forward, we focused on the core motifs. Their distribution across the subsets defined by \ac{TALE} protein binding was unexpected and reveals previously unnoticed specificities in TALE protein \textit{in vivo} site selection.

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{{{figures/coreMotifs.expected}}}
  \caption["Core" Motifs Expected in Peak subsets]{\textbf{"Core" Motifs Expected in Peak subsets.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
  \label{fig:coreMotifsExpected}
\end{figure}

Our \textit{a priori} expectation for the distribution of motifs, based on the extensive literature on \textit{in vitro} binding of the \ac{TALE} factors, is represented in Figure \ref{fig:coreMotifsExpected}. We expected the \ac{HEXA} motif to be prevalent in all subsets in which no Pbx1 was detected: \ac{meise}, \ac{prepe}, and \ac{mepr}. We expected the \ac{OCTA} motif to be present in all subsets with Pbx1 binding, and the DECA motif to be present in the intersection of Meis/Prep with Pbx1.

\begin{figure}[]
    \centering
  \includegraphics[width=\textwidth]{{{figures/coreMotifs.found}}}
  \caption["Core" Motifs Found in Peak subsets]{\textbf{"Core" Motifs Found in Peak subsets.} Sequence logos (\cite{Schneider1990}) for each core motif found in each peak subset. In a sequence logo, the height of a letter is proportional to its frequency at that position and the height of the total stack at each position is proportional to the invariance at that position in the original alignment.}
  \label{fig:coreMotifsFound}
\end{figure}

However, the actual results (summarised in Figure \ref{fig:coreMotifsFound}) diverged from our expectations. We found that the \ac{OCTA} motif is present in all Meis-positive subsets, included those that are Pbx1-negative. The \ac{DECA} motif, conversely, is present in all Prep-positive subsets, even in those that show no evidence of Pbx1 binding. \ac{HEXA} appears in both \ac{meise} and \ac{mepb}, but not in any Prep-positive subset. 

\subsection{Motif Frequencies}
% [MT: Yo este apartado lo pondría antes, como 4.2.4 (antes de Accesory Motifs). Teniamos ademas la significación de la  presencia de los motivos, no?  habría que incluirlo y señalar de alguna manera en la gráfica qué presentcias no son significativamente diferentes de lo esperado por azar.]
Motif discovery is a form of undirected search. Once having the motif definitions, we performed a directed search using the FIMO program from the MEME suite. Directed search can reveal the presence of motifs in subsets were they are present but not frequent enough to be detected by motif discovery. It also allows us to quantify motif frequencies.

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/barChart_coreMotifFreqs}
  \caption[Core Motif Frequency in Peak Subsets]{\textbf{Core Motif Frequency in Peak Subsets.} Percentage of peaks in each factor binding category that contain each core motif. ALL = total non-redundant genomic sites. Pr = \ac{prepe}, M = \ac{meise}, P = \ac{pbxe}, Pr-M = \ac{mepr}, Pr-P = \ac{pbpr}, M-P = \ac{mepb}, Pr-M-P = triple peaks.}
  \label{fig:coreMotifFreqs}
\end{figure}

The results are represented in Figure \ref{fig:coreMotifFreqs}. \ac{HEXA} is most frequent in triple peaks, but it is not the most frequent core motif in any peak subset. \ac{OCTA} is the most frequent core motif in \ac{meise} peaks, in triple peaks and in \ac{mepb} peaks, where it is present in over 50\% of the peaks. The special abundance of OCTA motif in \ac{mepb} reinforces the idea that these sites may represent Pbx-Hox binding sequences.  \ac{DECA} and \ac{DECAEXT} are very frequent in all subsets where Prep is involved. This is espcially marked in \ac{pbpr} peaks, where \ac{HEXA} and \ac{OCTA} are present in under 10\% of peaks but  \ac{DECAEXT} is present in over 70\% of the peaks. In general, Meis peaks tend to be \ac{OCTA}-positive and to a lesser extent \ac{HEXA}-positive and Prep peaks tend to be \ac{DECA}- or \ac{DECAEXT}-positive, and this tendencies are exacerbated by co-binding with Pbx1. 

\subsection{EMSA on Core Motifs}

To determine whether the identified core motifs are indeed capable of being bound by the \ac{TALE} proteins tested, we performed \ac{EMSA} experiments on them. In \ac{EMSA}, the electrophoretic mobility of a radioactively labelled DNA probe is measured in the absence and presence of purified proteins or a nuclear extract. A change in mobility indicates protein binding to the probe. If an undefined protein mixture such as a nuclear extract is used, super-shift experiments can be performed in which a specific antibody will further shift or inhibit the complex if it recognizes its target.

\begin{SCfigure}[]
  \centering
  \includegraphics[width=.6\textwidth]{figures/coreMotifEMSA}
  \caption[EMSA Experiments on Core Motifs]{\textbf{EMSA Experiments on Core Motifs.} leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda leyenda}
  \label{fig:coreMotifEMSA}
\end{SCfigure}

The result of \ac{EMSA} experiments on core motifs is shown in Figure \ref{fig:coreMotifEMSA}. \ac{EMSA} on probes from selected peaks showed that while Prep and Pbx can bind any of the core motifs identified, Meis can bind the HEXA and OCTA sequences but can only weakly bind to the DECA sequence.

\subsection{Hox Peak Overlap}

Only two ChIP-seq vertebrate Hox protein binding datasets have been published that use antibodies against the endogenous proteins rather than tagged versions of them (\cite{Jung2010,Donaldson2012}. These studies characterized binding of HoxC9 in E13.5 embryonic mouse spinal cord (\cite{Jung2010}) and of HoxA2 in E11.5 mouse embryo branchial arches (\cite{Donaldson2012}).

Since the motifs found in our peaks suggest a pre-eminent role for Meis as a Hox cofactor, we took advantage of the fact that the samples used are comparable to ours to contrast our datasets to the ones published. The results are summarised in Figure \ref{fig:HoxOverlap}.

We found that, of the \ac{TALE} factors we tested, Meis is the one that overlaps the most with known Hox \ac{GW} binding site datasets. 28.9\% of our Meis peaks overlap with either HoxA2 peaks, HoxC9 peaks, or both. The figures for Prep and Pbx1 are 8.1\% and 12.6\%, respectively. Moreover, the majority of Prep-Hox and Pbx1-Hox peaks are also Meis peaks: 185/271 (68.3\%) for Prep and 348/440 (79\%) for Pbx1. 

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/vennDiagram_A2C9overlap}
  \caption[Hox Peak Overlap]{\textbf{Hox Peak Overlap.} Overlap between Meis (red), Prep (blue) or Pbx1 (green) peaks and HoxA2(grey) and HoxC9 (yellow) peaks. Areas of the circles are proportional to total peak numbers. Intersection areas are roughly proportional to the number of common peaks.}
  \label{fig:HoxOverlap}
\end{figure}

Surprisingly, Meis peaks bearing the \ac{OCTA} motif are not much more likely to overlap with Hox peaks than those that do not contain it: 842/2467 (34.1\%) versus 801/3219 (24.9\%), although this difference is significant ($\chi^2$ p-value $= 2.46 x 10^{-14}$). In contrast, Pbx1 peaks with the \ac{OCTA} motif are much more likely to overlap Hoxes: 245/554 (44.2\%) versus 195/2950 (6.6\%) ($\chi^2$ p-value $= 1.04 x 10^{-132}$). The same is true of Prep peaks: 116/286 (40.6\%) \ac{OCTA}-positive peaks overlap Hoxes, versus 155/3045 (5.1\%) that do not ($\chi^2$ p-value $= 1.04 x 10^{-97}$).% [MT: Hoxa9 puede unirse directamente con Meis?, si es así y la secuencia a la que se une está descrita, podríamos buscar si en los comunes Meis-Hoxc9 detectamos esa secuencia.  tb podrias mirar si OCTA se asocia mucho mas en los Meis-A2 que en los Meis-C9]

These data support the notion that most Meis peaks represent actual targets of Hox binding and suggests that a subset of these could represent Meis-Hox interactions in the absence of Pbx. Since binding sites of only two of the 39 Hox proteins in the mouse embryo cover almost 30\% of Meis peaks despite the samples not being exactly equal, it is reasonable to hypothesise that additional Hox \ac{GW} DNA-binding datasets will eventually reveal most Meis sites as Hox DNA binding sites. We have not seen a clear association of Meis binding sites with HoxC9 rather than HoxA2, despite the fact that Meis can bind HoxC9 but not HoxA2 directly. In addition to this, the \ac{OCTA} is more abundant than \ac{HEXA} in Meis peaks, suggesting that Meis binding to Hox through Pbx interaction without itself binding DNA is the most frequent mode of meis interaction with Hoxes.

They also suggest that the most common cofactor for Hox proteins in the embryo is actually Meis, rather than Pbx. This is surprising since only Hox proteins in paralog groups 9 to 13 have been described to bind Meis directly, so other Hox proteins would depend on Pbx for Meis complex formation. 

\subsection{OCTA Motif Variants}

The \ac{OCTA} motif, which we hypothesize represents binding sites for Hox proteins, is an eight-nucleotide sequence with high variability in positions 5 and 6. Some of the 16 possible matches have been shown to be able to bind Hoxes. Specific variants are known to be preferentially bound \textit{in vitro} by 3' Hoxes (TGATTGAT), middle Hoxes (TGATTAAT) or more 5' Hoxes (TGATTTAT) (\cite{Shen1997a,Slattery2011}).

To study the correlation between \ac{TALE} protein binding and \ac{OCTA} motif variants, we quantified the representation of each of the 16 possible combinations of nucleotides at positions 5 and 6. To do this, we calculated the expected number of occurrences based on the nucleotide composition of the peak subset. Over-representation is thus the ratio of actual motif instances found by literal string matching to this expected number, for each \ac{OCTA} variant and peak subset. Results are represented in Figure \ref{fig:OCTAvariants}. [references for described Hox binding] 

\begin{figure}[]
  
  \centering
  \includegraphics[width=\textwidth]{figures/barChart_OCTAvariants}
  \caption[OCTA Motif Variants]{\textbf{OCTA Motif Variants.} Fold overrepresentation of A/TGATNNAT variants in peak subsets. Asterisks mark values of overrepresentation over 5 with Bonferroni adjusted p-values under $10^{-3}$.}
  \label{fig:OCTAvariants}
\end{figure}

We see a clear pattern of over-representation. Most \ac{OCTA} motif variants that have been described to be bound by Hoxes are over-represented in the non-redundant list of all peaks. Exceptions to this rule are WGATAGAT, WGATGCAT, and WGATCGAT. [are these two actually known to bind?]. In contrast, all variants not described to be bound by Hoxes are not significantly over-represented except WGATGAAT. % [MT: Es mejor poner sono el dinucleotido, lo demás solo estorba]

We do not see or see very little enrichment over random expectation of \ac{OCTA} variants in the subsets in which Meis is not involved (\ac{prepe}, \ac{pbxe}, \ac{pbpr}). The only clear exception to this is the WGATTGAT variant in \ac{pbpr} peaks. This lack of \ac{OCTA} enrichment is especially surprising for \ac{pbxe} peaks, which we expected would represent majoritarily Pbx-Hox binding.

In contrast, all subsets in which Meis is involved show marked enrichment of at least some \ac{OCTA} variants. In \ac{meise} peaks, the most enriched variant is WGATTTAT, which appears over 30 times more frequently than would be expected by chance and is described in the literature as the preferred binding sequence for Hox paralog groups 6 to 10 in combination with Pbx. After WGATTTAT, the next \ac{OCTA} variants most enriched in \ac{meise} peaks are WGATTGAT and WGATGGAT. WGATTGAT is the preferred binding target for Hox paralog groups 1 to 5 in combination with Pbx \textit{in vitro}. WGATGGAT has been described as a binding site in vivo in the HoxB1 rhombomere 4 \ac{ARE} repeat 3 [check, refs]. Also enriched in \ac{meise} peaks are WGATAAAT, WGATTAAT (preferred target for paralogs 3 to 7) and WGATGAAT. 

\ac{mepr} peaks show a pattern of enrichment similar to \ac{meise} peaks, but with higher levels of over-representation. The exception is the WGATAGAT variant, which is over-represented in \ac{mepr} but not \ac{meise} peaks. 

The pattern of \ac{OCTA} over-representation in \ac{mepb} peaks is markedly different from that of \ac{meise} peaks. General levels of enrichment are also higher. In these, the most enriched variant is WGATTGAT, appearing over 55 times more frequently than would be expected by chance. The next most enriched variants are WGATGGAT and WGATTTAT, and then WGATTAAT. In this subset, WGATAGAT is not significantly enriched. A number of undescribed variants are enriched at moderate levels that are not over-represented in any other subset. 

The over-representation pattern and levels in triple peaks are intermediate between those of \ac{mepb} and \ac{mepr} peaks. 

These results strongly suggest that many Meis peaks are potentially Hox targets. If they are indeed, our data suggest that, following \cite{Shen1997}, the preferred Hox paralog groups for formation of ternary Pbx1-Meis-Hox are first groups 1 to 5, then 6 to 10, and then 3 to 7. However, the relative frequency of specific Pbx-Hox target sequences in our data might be explained by the relative abundance of Hox proteins in the input sample.


\section{Transcriptional targets}

In order to identify correlations between \ac{TALE} factor binding and transcriptional activity in the embryo, we obtained RNA from E11.5 Meis1 \ac{KO} embryos, \ac{prepi} (see \ref{chp:intro}) embryos and \ac{WT} littermates. 

We found 855 up-regulated and 631 down-regulated genes in \ac{prepi} embryos, and 210 up-regulated and 198 down-regulated genes in Meis1 \ac{KO} embryos. 

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/barChart_changedGenes}
  \caption[Misregulated Genes in Meis1/Prep1-Deficient Embryos]{\textbf{Misregulated Genes in Meis1/Prep1-Deficient Embryos.} Left: number of genes up- or down-regulated in Meis1 \ac{KO} and \ac{prei} embryos, and coincidence between sets. Right: over-representation of co-regulated genes over random expectation.}
  \label{fig:changedGenes}
\end{figure}

\subsection{Co-Regulation}

To estimate the level of co-regulation between Meis and Prep proteins, we compared the number of genes that are mis-regulated in both kinds of defective embryos, either in the same direction or in oposite directions. 

The chance that a given gene is misregulated in Meis1-deficient embryos is about 5 times higher if it is misregulated in Prep1-deficient embryos and vice-versa, as shown in Figure \ref{fig:changedGenes}. All categories of co-regulation were over-represented compared to random expectation. The most over-represented category, with 10 times more genes than would be expected if the gene sets were independent, is composed of genes that are up-regulated in \ac{prepi} embryos and down-regulated in Meis1 \ac{KO} embryos, i.e., genes activated by Meis1 and repressed by Prep1.

\subsection{Gene-Peak Correlation}

We tried to establish how nearby binding events could be affecting the expressing of misregulated genes. In order to do that, we calculated the Meis and Prep peak density in \ac{TSSA}, \ac{IG} and \ac{CI} regions near misregulated genes and compared it to the corresponding \ac{GW} densities. The ratios are represented in Figure \ref{fig:coRegulation}.

\begin{SCfigure}[]
  \centering
  \includegraphics[width=.6\textwidth]{figures/barChart_correlationGenes}
  \caption[Correlation Between Misregulation and TALE Binding]{\textbf{Correlation Between Misregulation and TALE Binding.} Ratio of Meis and Prep peak density in misregulated gene-associated \ac{TSSA}, \ac{IG} and \ac{CI} regions to the corresponding \ac{GW} averages. Asterisks mark p-value \< $10^{-3}$.}
  \label{fig:coRegulation}
\end{SCfigure}

We found some significant correlations between \ac{TALE} factor binding near genes and misregulation of those genes in \ac{TALE} factor mutants. Binding of Meis near the \ac{TSS} of a gene is associated to the gene being up-regulated in the Meis \ac{KO}, i.e., repressed by Meis in normal conditions. Surprisingly, binding of Prep to \ac{IG} or \ac{CI} regions is also associated to up-regulation in Meis1 \ac{KO}, although the effect is smaller in magnitude. 

Genes down-regulated in the \ac{prepi} embryo have a higher than average probability to show Prep peaks in their promoter, and Meis or Prep peaks within the transcriptional unit. 

We could not find association between binding in any region and down-regulation in Meis1 \ac{KO} or up-regulation in \ac{prepi}. It has to be noted that we did not knock-out the full complement of Meis nor Prep genes in the embryo, so the weak correlations between protein binding and transcriptional effect we report could be due to redundancy with the rest of Meis/Prep factors. It also could mean that the relationship is more indirect than a simple binding-activation/repression model, or that we could not link the protein binding events to their actual target genes. 

\subsection{Gene Ontology}

Since Prep or Meis binding in the \ac{TSSA} or \ac{IG} regions of a gene might be associated to its transcriptional control by these factors, we performed Gene Ontology analysis on all genes bound by Meis or Prep to gain insight into their general functions. 

\begin{SCfigure}[]
  \centering
  \includegraphics[width=.7\textwidth]{figures/GO}
  \caption[Gene Ontology Categories Over-Represented in TALE-bound genes]{\textbf{Gene Ontology Categories Over-Represented in TALE-bound genes.} We restricted our search to the Biological Process ontology. We hand-selected the categories that were informative, i.e., specific and non-redundant. P-value cutoff was $10^{-5}$. Categories sorted by significance.}
  \label{fig:GO}
\end{SCfigure}

For both Meis- and Prep-bound genes, the most significant category was regulation of transcription. That supports the view that the \ac{TALE} proteins are master regulators in the sense that they regulate the transcription of other regulators (\acp{TF}) rather than effector genes. 

Most other over-represented categories were consistent with known roles of Meis1 (Nervous System Development, Heart Development, Embryonic Limb Morphogenesis, Blood Vessel Development) or Prep1 (DNA Metabolic Process, Cell Cycle, In Utero Embryonic Development) and generally suggested a framework in which Meis1 has many functions in specific developmental programs while Prep1 is more involved in basal cellular processes.


\section{Hox Clusters}

The Hox clusters are a genomic region of particular interest for the study of \ac{TALE} factor binding. 

\subsection{Distribution in the Hox clusters}

We studied the read distribution in the Hox clusters (Figure \ref{fig:readDensity}). In all 4 Hox clusters, Meis read density is much higher than in the surrounding regions, and shows a concentration near the 3' end that extends to paralog 9 but no further. [describir 3' far peaks]

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/readDensity_HoxClusters}
  \caption[Read Density in the Hox Clusters]{\textbf{Read Density in the Hox Clusters.} Meis ChIP-seq read density in the four mouse Hox Clusters, represented as bar graphs.}
  \label{fig:readDensity}
\end{figure}

\subsection{Peaks in the Hox Clusters}

Plotting Meis, Prep and Pbx1 peaks in the Hox clusters (Figure \ref{fig:peaksClusters}) reveals a clear restriction of \ac{TALE} protein binding events in our sample to the region 3' to paralog 9 in every cluster. 

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/peaks_HoxClusters}
  \caption[TALE Factor ChIP Peaks in the Hox Clusters]{\textbf{TALE Factor ChIP Peaks in the Hox Clusters.} Arrowheads represent peaks, black boxes represent transcriptional units. We have represented the short isoform of each paralog 3 for easier orientation.}
  \label{fig:peaksClusters}
\end{figure}

We found abundant Meis peaks in the Hox clusters (20 in HoxA, 13 in HoxB, 7 in HoxC and 8 in HoxD) and many fewer Prep or Pbx1 peaks (4, 4, 2, and 1 for Prep; 8, 4, 2 and 0 for Pbx1). The HoxA cluster is clearly more bound by the \ac{TALE} Hox cofactors than the rest. All but one Prep and Pbx1 peaks in the Hox clusters overlap with Meis. 

It is notable that the border of Meis binding versus non-binding is located precisely at the 9\textsuperscript{th} paralog. It is one of the three only paralog groups conserved in all four murine Hox clusters (the others being 4 and 13) and marks the transition between non-AbdB-related Hoxes and AbdB-related Hoxes. 

\subsection{Hox Regulatory Sites from the Literature}

Several auto- and cross-regulatory \ac{TALE} protein \acp{BS} have been described in the Hox clusters (\cite{Lampe2008, Gould1997, Jacobs1999, Manzanares2001, Poepperl1995, Tuempel2007}). 

Of these 6 described regulatory sites, 4 show Meis peaks. 3 of the 4 also show Pbx1 binding, and one of these three shows triple a Meis-Prep-Pbx1 triple peak. All of them contain \ac{OCTA} motifs. In three of the six cases \ac{HEXA} motifs are present at a short distance to the \ac{OCTA} sequence.

\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/peaks_RegSites}
  \caption[Previously Described Hox Regulatory Sites]{\textbf{Previously Described Hox Regulatory Sites.} In each panel, the top section represents the genomic neighbourhood (15kb), with peak positions marked by arrowheads as in Figure \ref{fig:peaksClusters}. The bottom section shows a close-up (1kb) on the described regulatory site, with motif instance sequences marked at their position and peaks represented as solid bars covering their full span. Black bars represent exons, gray bars represent introns or intergenic regions.}
  \label{fig:RegSites}
\end{figure}

\subsection{Hox RNAseq}

The subdivision of the Hox clusters regarding Meis is not restricted to DNA binding. We plotted transcriptional variations in Hox genes in the absence of Meis1 or Prep1 from our RNAseq data. 

While most of the transcriptional changes were too subtle to be significant individually, the pattern is consistent across clusters and significant (p-value \< 0.5). In the absence of Meis1, 3' Hoxes tend to be overexpressed while 5' Hoxes tend to be repressed. The opposite pattern can be seen when considering the expression of Hox genes in \ac{prepi} embryos: 3' Hoxes are generally very slightly repressed while 5' Hoxes are markedly overexpressed.

\begin{SCfigure}[]
  \centering
  \includegraphics[width=.7\textwidth]{figures/barChart_HoxVariation}
  \caption[Differential transcriptional response of the 3' and 5' halves of the Hox clusters]{\textbf{Differential transcriptional response of the 3' and 5' halves of the Hox clusters.} Meis \ac{KO} (red) or \ac{prepi} (blue) Hox transcripts abundance relative to \ac{WT} littermate controls, as percentages.}
  \label{fig:HoxTranscript}
\end{SCfigure}


This pattern is surprising in at least two ways: Meis is more co-expressed with 3' Hoxes than with 5' Hoxes, so our expectation was that its lack would inhibit 3' gene expression while maybe derepressing 5' Hoxes. At the same time, the largest effect we detect in the transcription of the Hox genes is the overexpression of 5' Hoxes in the absence of Prep1, a region where neither Prep nor Meis bind.

\subsection{AP ChIP}

The Meis peaks in a given Hox cluster could be occupied by Meis constitutively or they could be occupied alternatively in different cells. Since our initial approach can not distinguish between these two possibilities, we performed \ac{CHIP} in embryo fragments. We dissected embryos along the \ac{AP} axis and performed Meis and unspecific IgG \ac{CHIP} on the resulting nuclear extracts.

\begin{figure}[p]
  \centering
  \includegraphics[width=\textwidth]{figures/APchIP}
  \caption[ChIP on Antero-Posterior Embryo Fragments]{\textbf{ChIP on Antero-Posterior Embryo Fragments.} Band intensity of rate-limited PCR on the Meis ChIP products with peak primers. Numbers denote Meis peak number. For each peak, the left three lanes are PCR products from the three inputs at the same DNA concentration. The right six lanes are PCR products of a fixed amount of control (IgG) or Meis ChIP DNA.}
  \label{fig:APchip}
\end{figure}

The resulting DNA was subjected to \ac{PCR} using primers designed for amplification of products centered around each of the Meis peaks in the HoxA cluster and some peaks in other clusters, spanning approximately 100bp around each summit. The PCR conditions were chosen so that the amount of product was roughly proportional to the input. The results are shown in Figure \ref{fig:APchip}. 

We discarded those peaks for which the IgG bands were of uneven intensity and compared the intensity of the bands for the rest of the peaks. As can be seen in Figure \ref{fig:APchipSummary}, Meis peaks occupancy is limited in the anteriormost part of thembryo, maximal in the middle section and skewed towards 5' peaks in the posteriormost part of the embryo.
% [MT: define better IgG controls]
\begin{figure}[]
  \centering
  \includegraphics[width=\textwidth]{figures/APchIP_summary}
  \caption[HoxA Cluster Meis Peak Antero-Posterior Occupancy]{\textbf{HoxA Cluster Meis Peak Antero-Posterior Occupancy.} Summary of the results in Figure \ref{fig:APchip}. All peaks in the HoxA cluster were tested for AP Meis occupancy. An absent arrowhead indicates no binding detected, a light-colored arrowhead indicates positive but not predominant binding compared to other embryo regions, and a dark-colored arrowhead indicates predominant binding}
  \label{fig:APchipSummary}
\end{figure}

[por colocar:

- comparacion detallada con Meis ChIP-seq hematop {Wilson et al. 20101, Huang et al. 2012} y meis eye {Bovolenta}
- Chip-proteomics
- EMSA on OCTA variants

]
